# memo

## Table of Contents
- [species](#species)
- [taxonomy](#taxonomy)
- [featuring](#featuring)
- [updates](#updates)
- [2018](#2018)
- [2017](#2017)
- [2016](#2016)
- [archaea](#archaea)
- [primer](#primer)
- [CAMI_challenge](#cami) Critical Assessment of Metagenome Interpretation
- [16S](#16s)
- [k-mer](#k-mer)
- [jsme2018](#jsme2018) Microbial lifestyle and genome signatures.
- [2018-06](#2018-06)
- [people](#people)
  - [duvallet](#duvallet) Claire Duvallet
- [prokaryote](#prokaryote)
- [replication](#replication)

----------
## species

https://twitter.com/digitalwright/status/1047618983497818114
Erik Wright on Twitter: "Is "species" a real thing for bacteria? @baobabbaum and I did a deep dive into thousands of bacterial genomes to figure it out. Turns out the answer is yes and no. Check it out in #BMCGenomics: https://t.co/COJRzSpbs5… https://t.co/7maLMhhTZX"
6:46 PM - 3 Oct 2018

http://kazumaxneo.hatenablog.com/entry/2018/09/14/141442
MinHashを使った高速なANI計算ツール fastANI - macでインフォマティクス

バクテリア（および他の微生物）が離散したクラスター（discrete clusters ）（species）を形成するか、または水平遺伝子伝達（HGT）の頻度が高く、ゆっくりと減衰するキネティクスのために、連続的な遺伝的多様性が観察されるかどうかである。Closely relatedな少数のゲノムに基づく研究は、genetic continuum（連続性）が優先される可能性があることを示している（例えば、ref.1）。他方、HGTは種間の境界を歪めるほど頻繁ではないかもしれない、あるいは同じ種内の生物が種間の生物に比べてより頻繁に遺伝子交換を行い、それによって異なるクラスターを維持するとの研究がある（例えばref.2）。

種の境界を評価する際の1つの基本的な作業は、2つのゲノム間の遺伝的関連性の推定である。The whole-genome average nucleotide identity（ANI）は、同じ種に属する生物が典型的にはそれらの間に≥95％のANIを示すため（ref.3,4）、近年この作業のための堅牢な方法として浮上している。 

January 16, 2017
http://schaechter.asmblog.org/schaechter/2017/01/do-bacterial-species-really-exist-and-why-should-we-care.html
Small Things Considered: Do bacterial species really exist and why should we care?

----------
## taxonomy

http://gtdb.ecogenomic.org/
Genome Taxonomy Database

https://www.ncbi.nlm.nih.gov/pubmed/30148503
Nat Biotechnol. 2018 Aug 27. doi: 10.1038/nbt.4229. [Epub ahead of print]
A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life.
Parks DH1, Chuvochina M1, Waite DW1, Rinke C1, Skarshewski A1, Chaumeil PA1, Hugenholtz P1.

https://twitter.com/NatureBiotech/status/1034201413717450753
Nature Biotechnology on Twitter: "A standardized bacterial #taxonomy based on genome phylogeny substantially revises the tree of life https://t.co/VxV1FNxtAs #phylogenetics… https://t.co/AZR5RBc18E"
6:10 PM - 27 Aug 2018
![](https://pbs.twimg.com/media/Dlo37tsUYAAcnP8.jpg)

https://twitter.com/NatureRevMicro/status/1036595989816246272
Nature Rev Microbiol on Twitter: "#ResearchHighlight: Next-generation bacterial taxonomy https://t.co/Acx8vB0XuW"
8:45 AM - 3 Sep 2018

https://twitter.com/RPoretsky/status/1034446295451226113
Rachel Poretsky on Twitter: "The @PhilHugenholtz et al. paper that contributed to heated #ISME17 discussions is out! A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. https://t.co/lvWwHKS7OD"
10:23 AM - 28 Aug 2018

https://twitter.com/NatureRevMicro/status/1034357487355539456
Nature Rev Microbiol on Twitter: "A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life @NatureBiotech https://t.co/Hro7pynEPQ… https://t.co/ev7En4LKdL"
4:30 AM - 28 Aug 2018
![](https://pbs.twimg.com/media/DlrGnFvX4AA_a4L.jpg)

https://twitter.com/WvSchaik/status/1034188127261405185
Willem van Schaik on Twitter: "Exciting paper: 'A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life' by @PhilHugenholtz https://t.co/ABzGHUSuPn"
5:17 PM - 27 Aug 2018

https://twitter.com/Tyu_Shi/status/1034217003936931841
A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life https://www.nature.com/articles/nbt.4229 …  bioRxivに前に出ていたHugenholtz達の原核生物ゲノム系統樹論文が出たか。
7:12 PM - 27 Aug 2018


----------

## 2018

## 2018-09

## 2018-08


## 2018-07

https://twitter.com/mike_schatz/status/1017507044230823936
Registration is open for the Biological Data Science conference at @CSHL on Nov 7-10. Abstracts are due by Aug 18. Check out this awesome lineup of speakers and session chairs! #biodata18 @cshlmeetings @cncurtis @jtleek @BeEngelhardt https://meetings.cshl.edu/meetings.aspx?meet=DATA&year=18
4:32 PM - 12 Jul 2018

![](https://pbs.twimg.com/media/Dh7nYN0W0AA18_X.jpg)

----------
## 2017

https://www.ncbi.nlm.nih.gov/pubmed/29258574
Genome Biol. 2017 Dec 19;18(1):233. doi: 10.1186/s13059-017-1367-z.
MICRA: an automatic pipeline for fast characterization of microbial genomes from high-throughput sequencing data.
Caboche S1,2, Even G3,4, Loywick A3,4, Audebert C3,4, Hot D5,4.

http://nobukoba1988.hatenablog.com/entry/2017/12/10/144459
細菌が作り出す真核生物の交配を導く媚薬 〜その名はEroS〜 - 高等ムーミンをめぐる冒険

https://twitter.com/PLOSCompBiol/status/936576580742270981
PLOS Comp Biol on Twitter: "Congrats to @HusonDaniel and colleagues for winning our 2017 #PLOSCompBiol #ResearchPrize in the category Exemplary Methods/Software for their tool MEGAN CE that analyzes the genetic diversity of #microbiomes https://t.co/eBB8N4hNZk… https://t.co/ifKpnQ0voc"
7:43 AM - 1 Dec 2017

![](https://pbs.twimg.com/media/DP9jGWnUIAAFLW2.jpg)

https://twitter.com/Tyu_Shi/status/942824493285703680
Tyu_Shi on Twitter: "poplarとmaizeの根から単離されたバクテリア484株のゲノムを読んで、既知のゲノム3千株以上と比較ゲノム解析してplant-associated gene clustersを探索したのか。 https://t.co/DVNlG9j13e"
1:30 PM - 18 Dec 2017
https://www.ncbi.nlm.nih.gov/pubmed/24567124
Nat Genet. 2018 Jan;50(1):138-150. doi: 10.1038/s41588-017-0012-9. Epub 2017 Dec 18.
Genomic features of bacterial adaptation to plants.



https://twitter.com/MicrobiomDigest/status/936394562381029377
Elisabeth Bik on Twitter: "Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome Metagenomic study of microbial… https://t.co/sSXhDvasCH"
7:40 PM - 30 Nov 2017

https://twitter.com/thePeerJ/status/932675684543139845
PeerJ - the journal on Twitter: "PBxplore: a tool to analyze local protein structure and deformability with Protein Blocks https://t.co/p86EiqUZst #bioinformatics https://t.co/fJ1DjbVwFL"
1:23 PM - 20 Nov 2017

![](https://pbs.twimg.com/media/DPGHl2cW0AYCa1a.jpg)

https://twitter.com/Tyu_Shi/status/940987045790412800
Tyu_Shi on Twitter: "Metagenome Assembled Genomesの評価方法としてよく使われている、Single copy genesを用いたcompletenessとcontaminationの指標についての、コンパクトにまとまった良い説明だ。 https://t.co/rfYMlQba35"
11:49 AM - 13 Dec 2017

https://www.microbe.net/2017/12/13/why-genome-completeness-and-contamination-estimates-are-more-complicated-than-you-think/ 
Why genome completeness and contamination estimates are more complicated than you think – microBEnet: the microbiology of the Built Environment network


12:19 PM - 11 Dec 2017
https://twitter.com/NatureBiotech/status/940269831181557760
Nature Biotechnology on Twitter: "Metagenomic binning and association of plasmids with bacterial host genomes using DNA methylation https://t.co/Un1Mm1p561 #epigenomics… https://t.co/FCvLuBgoxb"



https://twitter.com/NatureMicrobiol/status/940566982742740992
Nature Microbiology‏ 
@NatureMicrobiol
 Dec 12
More
Current issue: #pangenome correspondences: 

1. Michiel Vos & @AdamEyreWalker: are #pangenomes adaptive? http://bit.ly/2nHM5DU  

2. @bjesseshapiro on applying population genetics: http://bit.ly/2nHM82y  

3. @jomcinerney @alanmcn1 & @Evol_Molly reply: http://bit.ly/2nHMaaG 


https://twitter.com/pathogenomenick/status/939889344453730304
"On the Problem of Sequence Leakage" by @OmicsOmicsBlog 
http://omicsomics.blogspot.co.uk/2017/12/on-problem-of-sequence-leakage.html …

Related: have noticed that students are confused that genome 'X' not in GenBank when they know it's been sequenced.
4 replies 13 retweets 29 likes
Reply  4   Retweeted  13   Like  29   Direct message

https://twitter.com/ASMicrobiology/status/939872479820111872
@ASMicrobiology
 Dec 10
More
WGS shows that one species is actually (at least) four. Increasing our concept of a microbial species has many implications, including in understanding infections http://ow.ly/PsSI30h5qLc 


https://twitter.com/Tyu_Shi/status/938098486280515584
Tyu_Shi on Twitter: "J. Bacteriol.がこの1-2年で出した、Chromosome & Gene expression regulation, Bacterial cell, Pathogenesis & Host interactionのminireviewのThematic collection、とても勉強になる。個人的には前2つが特に。　https://t.co/Lb68EKtbRb"

evolvability
https://twitter.com/mbiojournal/status/932941916211175424
mBio on Twitter: "Refactoring the genetic code for increased evolvability https://t.co/DUl2Jc3did… "

Curtis Huttenhowerラボの論文
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1359-z
Experimental design and quantitative analysis of microbial community multiomics
Genome Biology2017


1 Dec 2017
https://twitter.com/thePeerJ/status/936536355772993537
PeerJ - the journal on Twitter: "Kullback Leibler divergence in complete bacterial and phage genomes https://t.co/qN0QBSUXeR #Genomics https://t.co/CaMp8FbmKe"




https://www.ncbi.nlm.nih.gov/pubmed/29204318
PeerJ. 2017 Nov 30;5:e4026. doi: 10.7717/peerj.4026. eCollection 2017.
Kullback Leibler divergence in complete bacterial and phage genomes.
Akhter S1, Aziz RK2,3, Kashef MT2, Ibrahim ES2, Bailey B4, Edwards RA1,3,4,5.
Here, we calculated the Kullback-Leibler divergence from the mean amino acid content as a metric to compare the amino acid composition for a large set of bacterial and phage genome sequences. 

https://www.ncbi.nlm.nih.gov/pubmed/29038487
Nat Ecol Evol. 2017 Nov;1(11):1606-1615. doi: 10.1038/s41559-017-0340-2. Epub 2017 Oct 16.
The microbiome beyond the horizon of ecological and evolutionary theory.
Koskella B1, Hall LJ2, Metcalf CJE3.
horizontal gene transfer, the production of public goods, toxin and antibiotic production, rapid evolution, and feedbacks between the microbiome and its host

https://www.ncbi.nlm.nih.gov/pubmed/29021594
Nat Rev Microbiol. 2017 Oct 12;15(11):639-640. doi: 10.1038/nrmicro.2017.121.
Understanding drug resistance will improve the treatment of bacterial infections.
Piddock LJV1.

https://www.ncbi.nlm.nih.gov/pubmed/29018197
Nat Commun. 2017 Oct 10;8(1):841. doi: 10.1038/s41467-017-00808-w.
The chromosomal organization of horizontal gene transfer in bacteria.
Oliveira PH1,2, Touchon M3,4, Cury J5,6, Rocha EPC5,6.

https://www.ncbi.nlm.nih.gov/pubmed/28604660
Nat Biotechnol. 2017 Jul;35(7):676-683. doi: 10.1038/nbt.3886. Epub 2017 Jun 12.
1,003 reference genomes of bacterial and archaeal isolates expand coverage of the tree of life.



----------
## 2016

https://www.ncbi.nlm.nih.gov/pubmed/27541692
PLoS Biol. 2016 Aug 19;14(8):e1002533. doi: 10.1371/journal.pbio.1002533. eCollection 2016 Aug.
Revised Estimates for the Number of Human and Bacteria Cells in the Body.
Sender R1, Fuchs S2, Milo R1.

https://www.ncbi.nlm.nih.gov/pubmed/26867134
PLoS Comput Biol. 2016 Feb 11;12(2):e1004739. doi: 10.1371/journal.pcbi.1004739. eCollection 2016 Feb.
Practical Approaches for Detecting Selection in Microbial Genomes.
Hedge J1, Wilson DJ1,2.







----------
## archaea

https://twitter.com/microbeminded2/status/1011396711082700800
specific archaea-targeting methodology

https://www.biorxiv.org/content/early/2018/05/30/334748
Measuring the archaeome: detection and quantification of archaea signatures in the human body | bioRxiv

https://twitter.com/pathogenomenick/status/897061801090850819
"Universal" 16S primers miss a bunch of archaea:
7:46 AM - 14 Aug 2017

https://www.ncbi.nlm.nih.gov/pubmed/28251182
mSphere. 2017 Feb 22;2(1). pii: e00026-17. doi: 10.1128/mSphere.00026-17. eCollection 2017 Jan-Feb.
Unexplored Archaeal Diversity in the Great Ape Gut Microbiome.
Here we evaluate archaeal diversity in gut microbiomes using an approach that targets virtually all known members of this domain.

https://www.ncbi.nlm.nih.gov/pubmed/25144201
PLoS One. 2014 Aug 21;9(8):e105592. doi: 10.1371/journal.pone.0105592. eCollection 2014.
Development of a prokaryotic universal primer for simultaneous analysis of Bacteria and Archaea using next-generation sequencing.

https://www.researchgate.net/post/What_are_the_best_archaeal_and_universal_primers
Asked 4 years ago

https://www.researchgate.net/post/Can_you_suggest_Archaea_domain_specific_16S_primers_for_metagenomic_study_of_human_gut_flora
Asked 6 years ago

https://www.ncbi.nlm.nih.gov/pubmed/20940022
J Microbiol Methods. 2011 Jan;84(1):12-8. doi: 10.1016/j.mimet.2010.10.001. Epub 2010 Oct 19.
Novel primers for 16S rRNA-based archaeal community analyses in environmental samples.

----------
## primer

http://kazumaxneo.hatenablog.com/entry/2017/12/25/155719
特異的なプライマーを自動設計する Primer BLAST - macでインフォマティクス

http://togotv.dbcls.jp/20170511.html
2017-05-11 PCRプライマー設計用ツール Primer3の使い方 2017

2012年5月28日
http://first.lifesciencedb.jp/from_dbcls/e0001
統合遺伝子検索GGRNA：遺伝子をGoogleのように検索できるウェブサーバ : ライフサイエンス 新着論文レビュー
2．GGRNAの活用事例
・PCRのプライマー配列から増幅遺伝子や増幅領域を確認する　　CTAGCTGCCAAAGAAGGACAT comp:CAATGAGATGTTGTCGTGCTCのようにして，fowardプライマーの配列と，reverseプライマーの相補鎖の配列とを同時に検索すれば，PCRで増幅する遺伝子や増幅領域を確認できる．

http://staffblog.amelieff.jp/entry/2011/05/24/180000
誰でも簡単にPrimer設計！への道 ~Primer3について~

----------

https://github.com/hiromasaono/training/tree/master/180612_KeioSFC
今日から使える便利な生命科学系公共データベース・ウェブツール
小野 浩雅
2018年6月12日(火)

January 17, 2018
https://github.com/yuifu/ajacs68/blob/master/README.md
NGSデータから新たな知識を導出するためのデータ解析リテラシー
尾崎 遼 Haruka Ozaki

----------
## CAMI
Critical Assessment of Metagenome Interpretation

https://twitter.com/CAMI_challenge

https://msystems.asm.org/content/3/4/e00103-18
Critical Assessment of Metagenome Interpretation Enters the Second Round | mSystems

Currently, CAMI is preparing for a second round of challenges, tentatively planned to open later this year. CAMI will provide data sets representing different environments and again offer assembly, taxonomic and genomic binning, as well as taxonomic profiling challenges (Fig. 1). Two multisample “toy” data sets representing microbial communities from different human body sites and from mouse gut are already provided to allow participants to prepare for the challenges (https://data.cami-challenge.org/participate). 

![](https://msystems.asm.org/content/msys/3/4/e00103-18/F1.large.jpg)

https://www.ncbi.nlm.nih.gov/pubmed/28967888
Nat Methods. 2017 Nov;14(11):1063-1071. doi: 10.1038/nmeth.4458. Epub 2017 Oct 2.
Critical Assessment of Metagenome Interpretation-a benchmark of metagenomics software.


----------
## 16S

https://www.ncbi.nlm.nih.gov/pubmed/29506021
Bioinformatics. 2018 Jul 15;34(14):2371-2375. doi: 10.1093/bioinformatics/bty113.
Updating the 97% identity threshold for 16S ribosomal RNA OTUs.
Edgar RC1.

https://twitter.com/BioMickWatson

https://www.ncbi.nlm.nih.gov/pubmed/29427429
Appl Environ Microbiol. 2018 Mar 19;84(7). pii: e02627-17. doi: 10.1128/AEM.02627-17. Print 2018 Apr 1.
The Madness of Microbiome: Attempting To Find Consensus "Best Practice" for 16S Microbiome Studies.
Pollock J#1,2, Glendinning L#2, Wisedchanwet T2, Watson M2.

https://www.microbe.net/2018/03/03/the-attempt-to-find-best-practice-for-16s-studies/
The attempt to find “best practice” for 16S studies – microBEnet: the microbiology of the Built Environment network


https://www.ncbi.nlm.nih.gov/pubmed/29305502
Appl Environ Microbiol. 2018 Mar 1;84(6). pii: e00014-18. doi: 10.1128/AEM.00014-18. Print 2018 Mar 15.
How Much Do rRNA Gene Surveys Underestimate Extant Bacterial Diversity?
Rodriguez-R LM1,2,3, Castro JC1,2, Kyrpides NC4, Cole JR5,6, Tiedje JM5,7,6, Konstantinidis KT8,2,3.


https://twitter.com/search?q=ゲノム%2016S

https://twitter.com/Tyu_Shi/status/969218896371707904
16S rRNA遺伝子配列で85%以下のIdentityの場合には、近縁系統のゲノム内16S rRNA遺伝子コピー数を補正に使うと補正精度が悪くなるという結果を出し、それを根拠にCopyRighter等の既存の補正ツールを批判しているが、そんな遠い系統を基準に補正をしては精度が下がるのは当然だ。
9:32 AM - 1 Mar 2018

https://twitter.com/yokadzaki/status/968841333400944640
Yusuke Okazaki on Twitter: "16Sアンプリコンseqで各系統のrrnオペロンのコピー数の違いの補正に近縁系統の情報を使う方法はアテにならないので、補正しないで済むデータの使い方をしたほうがマシである、という論文"
8:32 AM - 28 Feb 2018

https://twitter.com/yokadzaki/status/968837626013794304
Yusuke Okazaki on Twitter: "この論文はBioRxivのコメント欄でやりあっていて面白いので併せて読んだほうがよい https://t.co/dq9i5so7XG"
8:17 AM - 28 Feb 2018

https://twitter.com/yokadzaki/status/949628412804481025
DB上の細菌ゲノムを使って、全ゲノム>95％で定義されるOTUと16S rRNA>97%で定義されるOTUがどれくらい乖離しているのかを調べた研究。その乖離度が系統によって大きく異なるという結果も（予想通りだけど）面白い。ゲノムサイズが関係ありそうな結果だけど、本文には言及が無かった
8:07 AM - 6 Jan 2018

https://twitter.com/yokadzaki/status/940465660852703232
ゲノムサイズが小さいもの同士でも「近縁系統とのゲノムの違いがどれくらい連続的か」というのに系統間で明らかな差があるというのは面白い。そしてこの結果は「16Sで見えていたものはバイアスだらけ」ということも意味している
1:17 AM - 12 Dec 2017

https://twitter.com/yokadzaki/status/936042447452323840
それを「種」と呼んでいいのかは分からないけど、全ゲノムの相同性で明確に他の系統と区別できる（中間的なものが存在しない）形でクラスターができるという報告が増えてきて、16Sで>97%とかいう謎の基準に依存した研究はもうじき過去のものになりそう
8:21 PM - 29 Nov 2017


https://www.ncbi.nlm.nih.gov/pubmed/28662636
BMC Genomics. 2017 Jun 29;18(1):499. doi: 10.1186/s12864-017-3888-y.
Relating genomic characteristics to environmental preferences and ubiquity in different microbial taxa.
Cobo-Simón M1, Tamames J2.

https://www.ncbi.nlm.nih.gov/pubmed/28705636
J Biotechnol. 2017 Nov 10;261:2-9. doi: 10.1016/j.jbiotec.2017.07.010. Epub 2017 Jul 10.
A review of bioinformatics platforms for comparative genomics. Recent developments of the EDGAR 2.0 platform and its utility for taxonomic and phylogenetic studies.
Yu J1, Blom J2, Glaeser SP3, Jaenicke S4, Juhre T4, Rupp O4, Schwengers O4, Spänig S4, Goesmann A4.
https://www.sciencedirect.com/science/article/pii/S0168165617315225
But, because it is well known that the phylogenetic resolution of the primary phylogenetic marker, the 16S rRNA gene, is limited at the lower taxonomic levels, the integration of a genome sequence based phylogeny into the polyphasic approach of the prokaryotic taxonomy and systematics is the clear consequence in the genome area (Chun and Rainey, 2014, Ramasamy et al., 2014, Oren and Garrity, 2014, Vandamme and Peeters, 2014, Sangal et al., 2016).

https://www.ncbi.nlm.nih.gov/pubmed/27603265
Genome. 2016 Oct;59(10):783-791. Epub 2016 May 11.
bcgTree: automatized phylogenetic tree building from bacterial core genomes.
Ankenbrand MJ1,1, Keller A1,1.

The need for multi-gene analyses in scientific fields such as phylogenetics and DNA barcoding has increased in recent years. In particular, these approaches are increasingly important for differentiating bacterial species, where reliance on the standard 16S rDNA marker can result in poor resolution. 

2017/08/30
https://www.aist.go.jp/aist_j/press_release/pr2017/pr20170830/pr20170830.html
産総研：進化系統分類の指標となる16S rRNA遺伝子の進化的な中立性を実験的に証明

https://www.nite.go.jp/nbrc/safety/risk_assessment.html
微生物利用におけるリスク評価に関する技術情報の提供 | バイオテクノロジー | 製品評価技術基盤機構
DDHにおける菌種判定の閾値である類似度70%は、ANIにおいて95-96%に相当すると報告されています。このことから、近年では、DDHの代替としてANIに基づく新種の提案が行われてきています(Baek et al. 2015, IJSEM 65: 504; Shahraki et al. 2015, IJSEM 65: 504)。
OTUとは、ある一定以上の類似性（一般的には96-97%）を持つ配列同士を一つの菌種のように扱うための操作上の分類単位です。

バイオレメディエーションにおける生態系影響評価手法	Illumina社MiSeqシーケンサーを用いた、環境中の微生物生態系解析の実験手法およびパイプラインツール「QIIME」を用いたデータ解析手法の手順書です。 2016年3月25日 https://www.nite.go.jp/data/000081956.pdf

https://seikagaku.jbsoc.or.jp/10.14952/SEIKAGAKU.2015.870475/data/index.html
Journal of Japanese Biochemical Society 87(4): 475-477 (2015)
16SリボソームRNAの水平伝播実験からみえてくるリボソームの可塑性
佃 美雪1,2，宮崎 健太郎1,3

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/52/2/52_70/_pdf
化学と生物 Vol. 52, No. 2, 2014
16S rRNA遺伝子の「水平伝播」
異種16S rRNAによる遺伝的相補

2014
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3903645/
MetaMetaDB: A Database and Analytic System for Investigating Microbial Habitability

Therefore, MetaMetaDB chooses the hits exhibiting above 97%, 95%, 90%, 85%, and 80% identity from the search result list, which correspond approximately to the taxonomic levels of species, genus, family, order, and class [1], respectively, 

02/02/13
http://www.jarmam.gr.jp/situmon/16sr_rna.html
16s rRNA の相同性と同定精度の関係は???
一般的には, 97%以上の相同性があれば類縁関係があり、99%以上であれば同種である可能性が高いとしています。
極端な例では, 100%の相同性が認められても, 属レベルで異なる例さえ示されているのです。

2012 年
https://www.jstage.jst.go.jp/article/jslab/23/1/23_24/_pdf
総説
16S rRNA 遺伝子の大量シークェンシングによる 菌叢解析の現状と問題点
是則 有希, Jiahui Jiang, 中山 二郎
属レベルまでの菌叢プロファイルであ れば、Ribosomal Database Project I(I 以下、RDP II)のウェ ブサイトにある RDP classifier を用いることで容易に菌叢 プロファイルに変換することができる。
一方、種レベルでの菌叢データを得ることは非常に困難 である。実際に、理論上 16S rRNA 遺伝子の部分配列か ら種を特定することは不可能であり、

2011
https://www.sbj.or.jp/wp-content/uploads/file/sbj/8912/8912_yomoyama_2.pdf
生物工学基礎講座 バイオよもやま話 何から始めよう 微生物の同定 細菌・アーキア編
浜田 盛之,鈴木 健一朗
特に 16S rRNA 遺伝子配列の相同値が 97% に満たない場合 などは新属になる可能性もあるため,系統樹を作成して 既知の属のクラスターに含まれるのかどうかを慎重に確 認したほうがよい.

https://www.mitsui-norin.co.jp/mmid/knowledge/yokota/index3.html
MMID｜微生物を知ろう｜微生物分類同定講座
16S ｒRNA遺伝子塩基配列の相同性が98％以上の場合、必ずしも同種であるとは限らず、属によっては99％以上であっても別種の場合があります。

----------
## k-mer

https://www.biorxiv.org/content/early/2018/04/19/304972
RefSeq database growth influences the accuracy of k-mer-based species identification | bioRxiv

https://twitter.com/aphillippy/status/987312716451807233
Adam Phillippy on Twitter: ""RefSeq database growth influences the accuracy of k-mer-based species identification" https://t.co/XSc6pYq6Wh A shame that @traingene didn't go with my suggested title "No, that's not actually anthrax on the subway, international space station, etc."… https://t.co/i6BH4NEswI"
8:51 AM - 20 Apr 2018

https://twitter.com/BioMickWatson/status/987217580648693760
Mick Watson on Twitter: "Never go to species, not with kmers, not with 16S. Neither are accurate enough (sorry!)… "
2:33 AM - 20 Apr 2018

----------
## jsme2018

http://meeting-jsme2018.com/symposium.html
日本微生物生態学会 第32回大会 シンポジウム
｢進化・群集生態学の新手法で切り拓く微生物研究のフロンティア｣
東樹 宏和（京都大学）、岩崎 渉（東京大学）

https://www.ncbi.nlm.nih.gov/pubmed/23024607
Curr Genomics. 2012 Apr;13(2):153-62.
Microbial lifestyle and genome signatures.
Dutta C1, Paul S.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3308326/
```INTRODUCTION
SEQUENCE FEATURES OF MICROBIAL GENOMES INFLUENCED BY LIFESTYLE
a) G+C-Content
b) Oligonucleotide Composition
c) Codon Usage
SPECIALIZED LIFESTYLES OF MICROBIAL COMMUNITY & THEIR GENOME SIGNATURESa) Obligatory Intracellular Lifestyle, Characterized by Genome Reductionb) Non-Specialized Intra-Amoebal Lifestyle, Characterized by Genome Expansionc) Strand-Specific Codon Bias, Frequently Observed in Intracellular Microorganismsd) Acquisition of Genome Islands in Organisms Having Pathogenic or Symbiotic Lifestylee) Discriminatory Genome Features of Trophic Life Strategies of Marine Microbesf) Distinct Genome Features in Microbes Adapted to Extreme Environmentsi) Microorganisms Thriving at High Temperaturesii) Microbes Thriving at High SalinityCONCLUDING REMARKS
```

b) Oligonucleotide Composition

It has later been reported [24] that the halophilic microbes, characterized by overrepresentation of the dinucleotides GA/TC, CG and AC/GT, can be differentiated from non-halophiles on the basis of their dinucleotide abundance values.

Free-living bacteria, in general, display stronger bias in oligonucleotide usage than host-associated bacteria, as observed in a hierarchical clustering based on hexanucleotide–based genome signatures of 867 prokaryotic genomes [31]. Recently, comparative analysis of tetranucleotide composition in a set of 774 sequenced microbial genomes revealed convergence of compositional patterns among genomes with similar habitats [34], displaying distinct clusters of obligate intracellular organisms (both pathogen and endosymbiont) and grouping of the halophilic bacterium Salinibacter rubber, not with its fellow Bacteroidetes, but with halophilic and methanogenic Archaea [34].

c) Codon Usage

Distinct niche-specific trends in synonymous codon usage have also been observed in microbes thriving at high salinity [24]. An analysis of synonymous codon usage patterns in bacterial and fungal genomes by Willenbrok et al. [49] demonstrated that differences in codon preferences of translational codon adaptation and dominant codon adaptation provide an environmental signature that can segregate bacteria according to their lifestyle, for instance soil bacteria and soil symbionts, spore formers, enteric bacteria, aquatic bacteria, and small intercellular and extracellular pathogens.

[24]
https://www.ncbi.nlm.nih.gov/pubmed/18397532
Genome Biol. 2008 Apr 9;9(4):R70. doi: 10.1186/gb-2008-9-4-r70.
Molecular signature of hypersaline adaptation: insights from genome and proteome composition of halophilic prokaryotes.
Paul S1, Bag SK, Das S, Harvill ET, Dutta C.

[31]
https://www.ncbi.nlm.nih.gov/pubmed/18421372
PLoS Comput Biol. 2008 Apr 18;4(4):e1000057. doi: 10.1371/journal.pcbi.1000057.
Investigations of oligonucleotide usage variance within and between prokaryotes.
Bohlin J1, Skjerve E, Ussery DW.

[34]
https://www.ncbi.nlm.nih.gov/pubmed/20333228
Genome Biol Evol. 2010 Jan 25;2:117-31. doi: 10.1093/gbe/evq004.
Distinguishing microbial genome fragments based on their composition: evolutionary and comparative genomic perspectives.
Perry SC1, Beiko RG.

[49]
https://www.ncbi.nlm.nih.gov/pubmed/17156429
Genome Biol. 2006;7(12):R114.
An environmental signature for 323 microbial genomes based on codon adaptation indices.
Willenbrock H1, Friis C, Juncker AS, Ussery DW.

----------
Microbial lifestyle and genome signatures.
C Dutta 著 - ‎2012 - ‎被引用数: 35

review
https://www.ncbi.nlm.nih.gov/pubmed/25610432
Front Microbiol. 2015 Jan 6;5:742. doi: 10.3389/fmicb.2014.00742. eCollection 2014.
Evolution of small prokaryotic genomes.
Martínez-Cano DJ1, Reyes-Prieto M2, Martínez-Romero E3, Partida-Martínez LP1, Latorre A2, Moya A2, Delaye L1.

review
https://www.ncbi.nlm.nih.gov/pubmed/24600039
Microbiol Mol Biol Rev. 2014 Mar;78(1):1-39. doi: 10.1128/MMBR.00035-13.
Bacterial genome instability.
Darmon E1, Leach DR.

review
https://www.ncbi.nlm.nih.gov/pubmed/24847957
Annu Rev Microbiol. 2014;68:117-35. doi: 10.1146/annurev-micro-091313-103436. Epub 2014 May 16.
What ecologists can tell virologists.
Dennehy JJ1.

----------
# Salinity

https://www.frontiersin.org/article/10.3389/fmicb.2018.01492
Frontiers | Microbial Community Structure and Functional Potential Along a Hypersaline Gradient | Microbiology

----------
ゲノムサイズとGC含量

https://twitter.com/figshare/status/406777844807639040
GC Content versus Genome Size - NCBI 2013 Completed Bacterial Genomes: http://figshare.com/articles/GC_Content_versus_Genome_Size_NCBI_2013_Completed_Bacterial_Genomes/865528 … #gc #content #genome
8:32 AM - 30 Nov 2013

![](https://pbs.twimg.com/media/BaUqS26CIAEw_AX.png)

http://asserttrue.blogspot.com/2013/05/dna-gc-content-and-survival-value.html
blogorrhea: DNA G+C Content and Survival Value

![](http://1.bp.blogspot.com/-wne70sOdrto/UY417aNwGfI/AAAAAAAABno/qCD9SK8hhLE/s400/GCchart.jpg)

https://www.ncbi.nlm.nih.gov/pubmed/19609354
PLoS Genet. 2009 Jul;5(7):e1000565. doi: 10.1371/journal.pgen.1000565. Epub 2009 Jul 17.
Origin of an alternative genetic code in the extremely small and GC-rich genome of a bacterial symbiont.
McCutcheon JP1, McDonald BR, Moran NA.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2704378/
Figure 1
Relationship between genome size and GC content for sequenced Bacterial and Archaeal genomes.

![http://schaechter.asmblog.org/schaechter/2009/07/an-iconoclastic-endosymbiont.html](http://schaechter.asmblog.org/.a/6a00d8341c5e1453ef0115723c61fc970b-300wi)

----------

review
https://www.ncbi.nlm.nih.gov/pubmed/28538166
Curr Opin Microbiol. 2017 Aug;38:74-80. doi: 10.1016/j.mib.2017.05.001. Epub 2017 May 21.
An evolutionary perspective on plasmid lifestyle modes.
Hülter N1, Ilhan J1, Wein T1, Kadibalban AS1, Hammerschmidt K1, Dagan T2.


https://www.tandfonline.com/doi/abs/10.1080/1040841X.2017.1332003
The multi-omics promise in context: from sequence to microbial isolate: Critical Reviews in Microbiology: Vol 44, No 2

https://www.tandfonline.com/doi/full/10.1080/1040841X.2017.1303661
The impact of insertion sequences on bacterial genome plasticity and adaptability: Critical Reviews in Microbiology: Vol 43, No 6

----------

https://twitter.com/yokadzaki/status/1016332354158071809
面白い。Pseudomonasに感染するファージのゲノムの多様性の高さに注目した研究。この結果を見ると、メタゲノムで得られた未知ウイルスゲノムのホストを特定するのは容易ではないし、比較的良い手がかりになるとされているnucleotide compositionにも実際はかなりばらつきがあるのが分かる
Translate Tweet
10:44 AM - 9 Jul 2018
https://www.frontiersin.org/articles/10.3389/fmicb.2018.01456/full
Frontiers | Comparative Genomic Analysis of 130 Bacteriophages Infecting Bacteria in the Genus Pseudomonas | Microbiology


----------
## 2018-06

https://twitter.com/DrKatHolt/status/1009959456849866752
Kat Holt on Twitter: "So many resistance genes... and now picking up KPC carbapenemase as well. But this ESBL/MDR clone has been spreading silently for a long time… https://t.co/brsd2djYxz"
8:41 PM - 21 Jun 2018

![](https://pbs.twimg.com/media/DgQXD5CU0AEbh15.jpg)

https://twitter.com/biorxivpreprint/status/1009741678289260544
bioRxiv on Twitter: "Emergence and rapid global dissemination of CTX-M-15-associated Klebsiella pneumoniae strain ST307 https://t.co/idRfAqGZhK #bioRxiv"

https://www.biorxiv.org/content/biorxiv/early/2018/06/21/352740.full.pdf
The distribution of ICEKp insertions on the ST307 core genome tree indicated >4 independent acquisitions with limited expansion of recipient sublineages (Figure 2), consistent with the patterns recently reported for ST258 and other common MDR Kp clones [19]. 

Figure 2: Bayesian phylogeny of 95 ST307 isolates. a, 
b, Presence of the yersiniabactin-carrying ICEKp elements (variants coloured as per inset legend), antimicrobial resistance genes (blocks coloured by drug class), and 


----------
## 2018-06-26

系統樹と遺伝子保存度（BSR値）

https://www.ncbi.nlm.nih.gov/pubmed/24749011
PeerJ. 2014 Apr 1;2:e332. doi: 10.7717/peerj.332. eCollection 2014.
The large-scale blast score ratio (LS-BSR) pipeline: a method to rapidly compare genetic content between bacterial genomes.
Sahl JW1, Caporaso JG2, Rasko DA3, Keim P1.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3976120/
The integrated toolkit for the exploration of microbial pan-genomes (ITEP) toolkit (Benedict et al., 2014) was recently published and performs similar functions to LS-BSR, including the identification of gene gain/loss at nodes of a phylogeny. 

Figure 2
The distribution of virulence factors and phylogenomic markers associated with a core single nucleotide polymorphism (SNP) phylogeny.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3976120/figure/fig-2/

系統樹と遺伝子の獲得・喪失

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3890548/
ITEP: An integrated toolkit for exploration of microbial pan-genomes

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3890548/figure/F2/
Figure 2
Illustration of ITEP’s capabilities for studying gene gain and loss patterns across a phylogeny. 

Analysis of gene gain and loss patterns across phylogeny

コアゲノム系統樹

https://www.ncbi.nlm.nih.gov/pubmed/22666370
PLoS One. 2012;7(5):e37607. doi: 10.1371/journal.pone.0037607. Epub 2012 May 30.
Gene repertoire evolution of Streptococcus pyogenes inferred from phylogenomic analysis with Streptococcus canis and Streptococcus dysgalactiae.
Lefébure T1, Richards VP, Lang P, Pavinski-Bitar P, Stanhope MJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/

ゲノムの数を減らして、オーソログの数を増やして、系統解析
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/figure/pone-0037607-g001/
Figure 1
Total evidence trees using the Streptococcus one-to-one core genes (701 orthologs, A) and the four species one-to-one core genes (1072 orthologs, B).

異なる樹形（トポロジー）の例
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3364286/figure/pone-0037607-g002/
Figure 2
The seven topologies and the number of genes mapped to them, found by the Bayesian gene tree concordance analysis.

#188ws 酵母23ゲノムの1070オーソログ遺伝子を用いた系統解析。1070遺伝子の連結配列concatenationに基づく種系統樹は、100%のbootstrapサポートを示すが、1070の異なる遺伝子系統樹の全てに一致しない http://www.ncbi.nlm.nih.gov/pubmed/23657258

http://d.hatena.ne.jp/haruosuz/20080813
1- ゲノムの遺伝子系統樹間の不一致 gene-tree incongruence

http://www.geocities.jp/ancientfishtree/phyMarker.html
系統推定 - Jun Inoue

Salichos L, Rokas A 2013. 
Inferring ancient divergences requires genes with strong phylogenetic signals. Nature 497: 327-+.

　酵母 23 種から得られた 1070 遺伝子 (シンテニーからオーソログと確認されている) を系統解析した研究では，どの遺伝子から得られた樹形も concatenate して得られた樹形とは異なっていた．

----------
## 2018-06-05

https://twitter.com/PatSchloss
https://www.mothur.org

https://twitter.com/Tyu_Shi/status/1006596459255377920
mothurの作者によるperspective。Identifying and Overcoming Threats to Reproducibility, Replicability, Robustness, and Generalizability in Microbiome Research　http://mbio.asm.org/content/9/3/e00525-18.full 
1:57 PM - 12 Jun 2018

https://twitter.com/lteytelman/status/1004254963541741568
This perspective by @PatSchloss should be required reading for scientists, funders, and pubilshers.  
"Identifying and Overcoming Threats to Reproducibility, Replicability, Robustness, and Generalizability in Microbiome Research" http://mbio.asm.org/content/9/3/e00525-18.full
2:53 AM - 6 Jun 2018

https://www.ncbi.nlm.nih.gov/pubmed/29871915
MBio. 2018 Jun 5;9(3). pii: e00525-18. doi: 10.1128/mBio.00525-18.
Identifying and Overcoming Threats to Reproducibility, Replicability, Robustness, and Generalizability in Microbiome Research.
Schloss PD1.

----------



----------
## people

## duvallet
Claire Duvallet

MITのEric Almラボ。微生物群集データのメタアナリシス。バッチ効果。

https://twitter.com/cduvallet

7:42 PM - 23 Apr 2018
https://twitter.com/cduvallet/status/988563768765505536
Claire Duvallet on Twitter: "Meaning to make a meta-analysis but blocked or bothered by batch effects? Use your controls as nulls: normalize features in cases relative to their distribution in control samples. Learn more at @PLOSCompBiol https://t.co/PHyVZoZlej Congrats @gibbological!! @ejalm"

1:55 PM - 23 Apr 2018
https://twitter.com/gibbological/status/988476547614105601
Sean Gibbons on Twitter: "Befuddled by batch effects? Check out our simple, non-parametric solution for 16S case-control studies in @PLOSCompBiol https://t.co/grMDm7SIQl -- method available as a @qiime2 plugin thanks to @cduvallet"

April 17, 2018
https://cduvallet.github.io/posts/2018/04/microbiome-data
An empirical analysis of microbiome data availability - Claire Duvallet

19 January 2018
https://onlinelibrary.wiley.com/doi/full/10.1111/1751-7915.13047
Meta‐analysis generates and prioritizes hypotheses for translational microbiome research - Duvallet - 2018 - Microbial Biotechnology - Wiley Online Library
Claire Duvallet

05 December 2017
https://www.nature.com/articles/s41467-017-01973-8
Meta-analysis of gut microbiome studies identifies disease-specific and shared responses | Nature Communications
Claire Duvallet, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry & Eric J. Alm

----------
## prokaryote

https://twitter.com/FenrirCappel/status/1037604131928592384
yuki_hamaguchi on Twitter: "真核生物か原核生物かわかっていない生物がいるらしい… しかも2012年に発見されてから、この１種以外見つかっていないみたい…不思議… https://t.co/BaANRqXCTo https://t.co/g4Dfwgu1tb"
3:31 AM - 6 Sep 2018

10-2-2017
https://ultrabem.com/organism/taxa/prokaryote.html
原核生物 Prokaryote: 核をもたない生物

2015年5月12日
http://aasj.jp/news/watch/3410
５月１２日：原核生物と真核生物をつなぐミッシングリンクの発見（Natureオンライン版掲載論文） | AASJホームページ

----------
# replication
複製

遺伝子がリーディング鎖に偏る。必須遺伝子で顕著
French, S. 1992. “Consequences of Replication Fork Movement through Transcription Units in Vivo.” Science (New York, N.Y.) 258 (5086): 1362–65. http://www.ncbi.nlm.nih.gov/pubmed/1455232.

Rocha, Eduardo P C, and Antoine Danchin. 2003. “Essentiality, Not Expressiveness, Drives Gene-Strand Bias in Bacteria.” Nature Genetics 34 (4): 377–78. doi:10.1038/ng1209.

Srivatsan, Anjana, Ashley Tehranchi, David M MacAlpine, and Jue D Wang. 2010. “Co-Orientation of Replication and Transcription Preserves Genome Integrity.” PLoS Genetics 6 (1): e1000810. doi:10.1371/journal.pgen.1000810.

Zheng, Wen-Xin, Cheng-Si Luo, Yan-Yan Deng, and Feng-Biao Guo. 2015. “Essentiality Drives the Orientation Bias of Bacterial Genes in a Continuous Manner.” Scientific Reports 5 (January): 16431. doi:10.1038/srep16431.

- Arakawa, Kazuharu, and Masaru Tomita. 2012. “Measures of Compositional Strand Bias Related to Replication Machinery and Its Applications.” Current Genomics 13 (1): 4–15. doi:10.2174/138920212799034749.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3269016/
First, coding sequences are preferentially located in the leading strand in most genomes [25], 

- https://www.ncbi.nlm.nih.gov/pubmed/27196606
PLoS One. 2016 May 19;11(5):e0154306. doi: 10.1371/journal.pone.0154306. eCollection 2016.
Codon Adaptation of Plastid Genes.
Suzuki H1, Morton BR2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4873144/figure/pone.0154306.g007/
Fig 7
Gene distribution by strand in plastid genomes.

- https://www.ncbi.nlm.nih.gov/pubmed/12217498
Trends Microbiol. 2002 Sep;10(9):393-5.
Is there a role for replication fork asymmetry in the distribution of genes in bacterial genomes?
Rocha E1.
Fig. 2. Distribution of the percentage of genes in the leading strand (a)

----------







