Haruo Suzuki (haruo[at]g-language[dot]org)  
Last Update: 2018-09

----------

# bacteria
**細菌**

## Table of Contents
- [antimony](#antimony)
- [arsenic](#arsenic)
- [halogen](#halogen)
- [iodide](#iodide)
- [Bifidobacterium](#bifidobacterium)
- [Clostridium](#clostridium)
- [Clostridium difficile](#clostridium-difficile)
- [Geobacter](#geobacter)
- [Pseudomonas](#pseudomonas)
  - [Pseudomonas aeruginosa](#pseudomonas-aeruginosa)
  - [Pseudomonas stutzeri](#pseudomonas-stutzeri)
- codon usage bias
- [Rhodococcus](#rhodococcus)
  - Br-6
  - DLD
  - Bromate Reduction
- [Sinorhizobium](#sinorhizobium)
  - KGO-5
- [Rickettsiales](#rickettsiales)
- [Wolbachia](#wolbachia)

----------
https://ja.wikipedia.org/wiki/第15族元素
周期表において第15族に属する窒素・リン・ヒ素・アンチモン・ビスマス・モスコビウムのこと。

## arsenic
https://ja.wikipedia.org/wiki/ヒ素

https://ja.wikipedia.org/wiki/亜ヒ酸
亜ヒ酸（あヒさん、arsenous acid、arsenious acid）は、化学式がAs(OH)3の無機化合物である。


亜ヒ酸酸化
https://www.ncbi.nlm.nih.gov/pubmed/29250936
Microbiologyopen. 2018 Jun;7(3):e00567. doi: 10.1002/mbo3.567. Epub 2017 Dec 17.
Expression of the arsenite oxidation regulatory operon in Rhizobium sp. str. NT-26 is under the control of two promoters that respond to different environmental cues.
Corsini PM1, Walker KT1, Santini JM1.

http://www.microbial-ecology.jp/meeting/JSME2016/pdf/P-170.pdf
日本微生物生態学会第 31 回横須賀大会2016 JSME annual meetingP-170土壌環境のヒ素挙動に関与する微生物群集機能の解明○濱村 奈津子1, 片岡 剛文2
亜ヒ酸 (AsIII)はヒ酸(AsV)と比べ土壌中での移動性および生体毒性が高いため、反応速度の高い微 生物代謝によるヒ素の形態変化は、環境挙動に影響し汚染リスクを悪化させる要因と考えられ ている。

https://www.ncbi.nlm.nih.gov/pubmed/26790947
FEMS Microbiol Rev. 2016 Mar;40(2):299-322. doi: 10.1093/femsre/fuv050. Epub 2016 Jan 19.
The microbial genomics of arsenic.
Andres J1, Bertin PN2.
https://academic.oup.com/femsre/article/40/2/299/2570226
Figure 1.
Filled circles, squares and triangles stand for organisms bearing aio, arr and arx genes, respectively. Open squares stand for organisms respiring As(V) in which the arr genes have not yet been clearly identified (see details in the text for P. calidifontis). 


https://www.nies.go.jp/kanko/news/26/26-5/26-5-03.html
微生物を利用して汚染土壌からヒ素を除去する （2007年度 26巻5号）｜国環研ニュース 26巻｜国立環境研究所

亜ヒ酸・アンチモン酸化細菌
https://www.ncbi.nlm.nih.gov/pubmed/23666539
Microbes Environ. 2013;28(2):257-63. Epub 2013 May 11.
Identification of antimony- and arsenic-oxidizing bacteria associated with antimony mine tailing.
Hamamura N1, Fukushima K, Itai T.

----------
## antimony
https://ja.wikipedia.org/wiki/アンチモン

https://www.ncbi.nlm.nih.gov/pubmed/28768219
J Hazard Mater. 2018 Jan 5;341:36-45. doi: 10.1016/j.jhazmat.2017.07.042. Epub 2017 Jul 21.
Removal of antimonate from wastewater by dissimilatory bacterial reduction: Role of the coexisting sulfate.
Zhu Y1, Wu M2, Gao N1, Chu W1, An N1, Wang Q1, Wang S1.
- The priority pollutant antimony (Sb) exists primarily as Sb(V) and Sb(III) in natural waters, and Sb(III) is generally with greater mobility and toxicity than Sb(V).
- , new genera of Pseudomonas and Geobacter emerged with large proportions in both Sb-fed and Sb-sulfate-fed sludge, 

https://www.ncbi.nlm.nih.gov/pubmed/27342551
Appl Environ Microbiol. 2016 Aug 30;82(18):5482-95. doi: 10.1128/AEM.01375-16. Print 2016 Sep 15.
Microbial Antimony Biogeochemistry: Enzymes, Regulation, and Related Metabolic Pathways.
Li J1, Wang Q1, Oremland RS2, Kulp TR3, Rensing C4, Wang G5.
- In this review, we summarize the recent research on bacterial antimony transformations, especially those regarding antimony uptake, efflux, antimonite oxidation, and antimonate reduction.

https://www.ncbi.nlm.nih.gov/pubmed/26519630
Water Res. 2016 Jan 1;88:467-474. doi: 10.1016/j.watres.2015.10.042. Epub 2015 Oct 23.
Autotrophic antimonate bio-reduction using hydrogen as the electron donor.
- a short rod-shaped phylotype of Rhizobium (α-Protobacteria) that may have been active in Sb(V) reduction.

既存のSb還元細菌
- Sinorhizobium sp. JUK-1
- Desulfuribacillus stibiiarsenatis MLFW-2

#### Sinorhizobium sp. JUK-1
https://www.tandfonline.com/doi/abs/10.1080/01490451.2014.901440
01 Apr 2014
Isolation and Characterization of Antimony-Reducing Bacteria from Sediments Collected in the Vicinity of an Antimony Factory
Van Khanh Nguyen & Jong-Un Lee
- An antimonate-respiring bacterium
- Antimonate (pentavalent antimony) was reduced to antimonite (trivalent antimony) 
- a new strain of the Sinorhizobium genus.

#### Desulfuribacillus stibiiarsenatis MLFW-2
https://www.ncbi.nlm.nih.gov/pubmed/27974090
Int J Syst Evol Microbiol. 2017 Apr;67(4):1011-1017. doi: 10.1099/ijsem.0.001732. Epub 2017 May 5.
Desulfuribacillus stibiiarsenatis sp. nov., an obligately anaerobic, dissimilatory antimonate- and arsenate-reducing bacterium isolated from anoxic sediments, and emended description of the genus Desulfuribacillus.
Abin CA1, Hollibaugh JT2.

https://www.ncbi.nlm.nih.gov/pubmed/24319985
Environ Sci Technol. 2014;48(1):681-8. doi: 10.1021/es404098z. Epub 2013 Dec 17.
Dissimilatory antimonate reduction and production of antimony trioxide microcrystals by a novel microorganism.
Abin CA1, Hollibaugh JT.
- a bacterium that is capable of using antimonate [Sb(V)] as a terminal electron acceptor for anaerobic respiration, resulting in the precipitation of antimonite [Sb(III)] as microcrystals of antimony trioxide. The bacterium, designated strain MLFW-2, is a sporulating member of a deeply branching lineage within the order Bacillales (phylum Firmicutes).


https://www.ncbi.nlm.nih.gov/nuccore/MJAT00000000
Desulfuribacillus stibiiarsenatis strain MLFW-2, whole genome shotgun sequencing project

	$grep "MLFW-2" assembly_summary_refseq.txt 
	GCF_001742305.1	PRJNA224116	SAMN05771867	MJAT00000000.1	representative genome	1390249	1390249	Desulfuribacillus stibiiarsenatis	strain=MLFW-2		latest	Contig	Major	Full2016/09/22	ASM174230v1	University of Georgia	GCA_001742305.1	identical	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/742/305/GCF_001742305.1_ASM174230v1		assembly from type material
	
	$grep "MLFW-2" prokaryotes.txt 
	Desulfuribacillus stibiiarsenatis	1390249	PRJNA343013	343013	Terrabacteria group	Firmicutes	3.1197	38.2	-	MJAT01	40	3076	2935	2016/09/22	2017/04/13	Contig	University of Georgia	SAMN05771867	GCA_001742305.1	REFR	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/742/305/GCA_001742305.1_ASM174230v1	-	MLFW-2

#### kaken
	2015-04-01 – 2018-03-31
https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-15K14594/
KAKEN — 研究課題をさがす | 毒性元素の生物利用機構から解き明かす微生物エネルギー代謝の多様性進化 (KAKENHI-PROJECT-15K14594)
濱村 奈津子
ヒ素と同族の毒性元素であるアンチモンに着目し、平成27年度は高濃度のアンチモンで汚染した環境試料からアンチモン代謝微生物群の集積培養を試みた。
これまでに分離が報告されているアンチモン還元菌は2株のみであるが、本研究でアンチモン還元活性を示したコンソーシアからは、ヒ素やその他金属の還元菌と系統的類似度の高い多数のFirmicutes属の16S rRNA遺伝子配列とともに、アンチモン酸化および還元菌として報告のあるα-Proteobacteriaに近縁な配列が検出されたことから、これらの菌がアンチモン還元に関与している可能性が高い。

	2015-04-01 – 2018-03-31
https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-15K12230/
KAKEN — 研究課題をさがす | 微生物機能を利用した廃水からのアンチモン除去技術の基礎開発 (KAKENHI-PROJECT-15K12230)
	2015-04-01 – 2018-03-31
アンチモン還元菌の単離と特徴づけ

----------
## halogen

https://ja.wikipedia.org/wiki/第17族元素
halogen ハロゲン）は周期表において第17族に属する元素の総称。フッ素・塩素・臭素・ヨウ素・アスタチン・テネシンがこれに分類される。

https://www.genome.jp/dbget-bin/www_bget?ko:K01560	
2-haloacid dehalogenase [EC:3.8.1.2]

https://www.ncbi.nlm.nih.gov/pubmed/7944368
Appl Environ Microbiol. 1994 Sep;60(9):3375-80.
Comparative studies of genes encoding thermostable L-2-halo acid dehalogenase from Pseudomonas sp. strain YL, other dehalogenases, and two related hypothetical proteins from Escherichia coli.
Nardi-Dei V1, Kurihara T, Okamura T, Liu JQ, Koshikawa H, Ozaki H, Terashima Y, Esaki N, Soda K.

https://repository.kulib.kyoto-u.ac.jp/dspace/bitstream/2433/182417/1/ynogk01019.pdf
Catalytic Mechanism of L-2-Haloacid Dehalogenase from Pseudomonas sp.YL

https://pdbj.org/mom/220
220: 脱ハロゲン酵素 (Dehalogenases) - 今月の分子 - PDBj

最初の酵素は ハロアルカン脱ハロゲン酵素 (haloalkane Dehalogenases、PDBエントリー 2dhc )で、一方の塩素原子を取り除きヒドロキシ基と置き換える。次に別の2つの酵素がヒドロキシ基を酸に変換する(ここには示していない)。最後に ハロ酸脱ハロゲン酵素 (haloacid Dehalogenases、PDBエントリー 1jud )がもう一つの塩素原子を取り除く。

http://www.bio.nite.go.jp/mifup/wiki/index.php?title=ハロアルカン分解
ハロアルカンデハロゲナーゼ (EC 3.8.1.5) は α/β-hydrolase family に属する加水分解酵素で、アルコールとハロゲン化水素を生じる反応を触媒する。 生じたアルコールは alcohol dehydrogenase、および aldehyde dehydrogenase により haloacetate に変換され、haloacetate dehalogenase (EC 3.8.1.3) により glycolate に、haloalkanoic acid dehalogenase (EC 3.8.1.2) により 2-hydroxyacid に代謝される。

- NRULE_0051 (S)-2-haloacid dehalogenase
- NRULE_0075 Haloacetate dehalogenase

https://www.ncbi.nlm.nih.gov/pubmed/27347470
Springerplus. 2016 May 20;5(1):695. doi: 10.1186/s40064-016-2328-9. eCollection 2016.
l-2-Haloacid dehalogenase (DehL) from Rhizobium sp. RC1.
Adamu A1, Wahab RA2, Huyop F1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899344/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899344/table/Tab1/?report=objectonly
Table 1
Known haloacid-dehalogenating bacteria and their dehalogenases

P. putida PP3
Pseudomonas putida 109
Pseudomonas sp. 113
Pseudomonas sp. CBS3
P. putida strain AJ1
Pseudomonas fluorescens 1
Pseudomonas acidovoran
Pseudomonas sp. YL
Pseudomonas sp. R1
Pseudomonas sp. B6P
P. putida S3

Pseudomonas putida 109
Pseudomonas sp. 113
Pseudomonas sp. CBS3


https://link.springer.com/chapter/10.1007/978-3-662-06066-7_9
Bacterial Reductive Dehalogenases | SpringerLink
MB Habash 著 - ‎2004 - 
This chapter examines bacterial reductive dehalogenases that catalyze the substitution of a halogen substituent, typically a chlorine, bromine or iodine atom, with a hydrogen atom.

https://www.ncbi.nlm.nih.gov/pubmed/15171686
J Endocrinol. 2004 Jun;181(3):385-92.
Comparative kinetic characterization of rat thyroid iodotyrosine dehalogenase and iodothyronine deiodinase type 1.
Solís-S JC1, Villalobos P, Orozco A, Valverde-R C.
A distinct group of dehalogenases, the so-called iodothyronine deiodinases (IDs), that specifically remove iodine atoms from iodothyronines were subsequently discovered and have been extensively characterized.
The results reported here support the view that the rat thyroid gland has a distinctive set of dehalogenases specialized in iodine metabolism.

https://www.ncbi.nlm.nih.gov/pubmed/15289438
FASEB J. 2004 Oct;18(13):1574-6. Epub 2004 Aug 2.
Iodotyrosine dehalogenase 1 (DEHAL1) is a transmembrane protein involved in the recycling of iodide close to the thyroglobulin iodination site.
In the thyroid, iodotyrosine dehalogenase acts on the mono and diiodotyrosines released during the hydrolysis of thyroglobulin to liberate iodide, which can then reenter the hormone-producing pathways. 

https://www.ncbi.nlm.nih.gov/pubmed/10389261
Lett Appl Microbiol. 1999 Jun;28(6):445-7.
An easy colorimetric assay for screening and qualitative assessment of deiodination and dehalogenation by bacterial cultures.
Kuritz T1.
https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2672.1999.00538.x
 using this method for a variety of bacterial cultures, including Pseudomonas and Arthrobacter, with a variety of substrates. 

https://www.ncbi.nlm.nih.gov/pubmed/3624201
J Bacteriol. 1987 Sep;169(9):4049-54.
Purification and properties of haloalkane dehalogenase from Corynebacterium sp. strain m15-3.
Yokota T, Omori T, Kodama T.
https://www.ncbi.nlm.nih.gov/pubmed/7068529
J Bacteriol. 1982 May;150(2):522-7.
Purification and properties of a new enzyme, DL-2-haloacid dehalogenase, from Pseudomonas sp.
Motosugi K, Esaki N, Soda K.

----------
## iodide

https://ja.wikipedia.org/wiki/ヨウ素
iodine
ハロゲン元素の一つ。ヨード（沃度）ともいう。
反応性は塩素、臭素より小さい。水にはあまり溶けないが、ヨウ化カリウム水溶液にはよく溶ける。これは下式のように、ヨウ化物イオンとの反応が起こることによる。

ヨウ化物イオン

https://ja.wikipedia.org/wiki/ヨウ化物
iodide) イオンは、-1の電荷を帯びたヨウ素原子である[1]。
ヨウ素はヨウ化物水溶液によく溶け、茶色の三ヨウ化物イオンを形成する。

https://ja.wikipedia.org/wiki/三ヨウ化物
triiodide）は、主に3つのヨウ素原子による多原子アニオン、三ヨウ化物イオン (I3−) を含む塩を意味する。

ヨウ素酸イオン

https://ja.wikipedia.org/wiki/ヨウ素酸塩
iodate）は、ヨウ素酸イオンを含む塩である[1]。ヨウ素酸イオン (IO3-) は、ヨウ素原子に3個の酸素原子が結合した、三角錐形構造をとる。

https://ja.wikipedia.org/wiki/ヨウ素酸
iodic acid) とは、ヨウ素のオキソ酸の一種で、化学式 HIO3 の化合物。
水溶液中では、解離してヨウ素酸イオンを生じる。

http://www.wmsym.org/archives/2015/papers/15226.pdf
WM2015 Conference, March 15-19, 2015, Phoenix, Arizona, USA.Dissimilatory Iodate Reduction by Hanford Microbial Isolates – 15226Joshua T. Ellis*, Alex Dodwell*, Danielle Saunders*, Brady Lee*, and M. Hope Lee* *Pacific Northwest National Laboratory (USA)

An iodate reducing organism, designated as Agrobacterium tumefaciens strain AD35, was isolated from

----------

## Bifidobacterium

https://www.ncbi.nlm.nih.gov/pubmed/29222102
Appl Environ Microbiol. 2018 Jan 31;84(4). pii: e02249-17. doi: 10.1128/AEM.02249-17. Print 2018 Feb 15.
Tracking the Taxonomy of the Genus Bifidobacterium Based on a Phylogenomic Approach.
Lugli GA#1, Milani C#1, Duranti S1, Mancabelli L1, Mangifesta M2, Turroni F1,3, Viappiani A2, van Sinderen D4,5, Ventura M6,3.

https://www.ncbi.nlm.nih.gov/pubmed/28855614
Sci Rep. 2017 Aug 30;7(1):9926. doi: 10.1038/s41598-017-10546-0.
Predicting synonymous codon usage and optimizing the heterologous gene for expression in E. coli.
Tian J1, Yan Y1,2, Yue Q1,3, Liu X1, Chu X1, Wu N4, Fan Y1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5577221/
Figure 1
Clustering results of the codon usage pattern of different species. The row and column represent the codon usage pattern and the different bacterial subspecies.
 57 Bifidobacterium animalis, 58 Bifidobacterium longum, 

https://www.ncbi.nlm.nih.gov/pubmed/25842224
World J Microbiol Biotechnol. 2015 Jun;31(6):959-81. doi: 10.1007/s11274-015-1850-1. Epub 2015 Apr 5.
Comparative investigation of the various determinants that influence the codon and amino acid usage patterns in the genus Bifidobacterium.
Roy A1, Mukhopadhyay S, Sarkar I, Sen A.

----------

## Clostridium
- https://ja.wikipedia.org/wiki/クロストリジウム属

https://twitter.com/NatureRevMicro/status/1034357487355539456
Nature Rev Microbiol on Twitter: "A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life @NatureBiotech https://t.co/Hro7pynEPQ… https://t.co/ev7En4LKdL"
4:30 AM - 28 Aug 2018

![](https://pbs.twimg.com/media/DlrGnFvX4AA_a4L.jpg)

https://www.nature.com/articles/nbt.4229

This inconsistency is partly attributable to historical phenotype-based classification, as exemplified by the clostridia: microorganisms sharing morphological similarities have been erroneously classified in the genus Clostridium9,10.

The genus Clostridium was the most polyphyletic, representing 121 genera spanning 29 families, and was followed by Bacillus (81 genera across 25 families) and Eubacterium (30 genera across 8 families). 

The genus Clostridium is widely acknowledged to be polyphyletic, and efforts have been made to rectify this problem, including a global attempt to reclassify the genus by using a combination of phylogenetic markers9. The authors of that study have proposed the reclassification of 78 Clostridium species, and nine other species, into six novel genera9,52. Of these, we confirmed that Erysipelatoclostridium (with the exception of Clostridium innocuum str. 2959), Gottschalkia and Tyzzerella (excepting Clostridium nexile CAG:348) represent monophyletic genus-level groups. 

Similarly to its type genus, the order Clostridiales was extensively subdivided (Fig. 5a), largely as a consequence of an anomalous RED for this rank (Fig. 2a).

Figure 5: Comparisons of NCBI and GTDB classifications of genomes designated as Clostridia or Bacteroidetes in the GTDB taxonomy.

(a) Comparison of NCBI (left) and GTDB (right) order-level classifications of the 2,368 bacterial genomes assigned to the class Clostridia in the GTDB taxonomy. Genomes classified in a class other than Clostridia by NCBI are indicated in parentheses. 

9.
Yutin, N. & Galperin, M.Y. A genomic update on clostridial phylogeny: Gram-negative spore formers and other misplaced clostridia. Environ. Microbiol. 15, 2631–2641 (2013).


## Clostridium difficile
- https://ja.wikipedia.org/wiki/クロストリジウム・ディフィシル腸炎
- https://ja.wikipedia.org/wiki/クロストリジウム・ディフィシル

https://institute.yakult.co.jp/bacteria/4205/
クロストリジウム ディフィシル | 菌の図鑑 | ヤクルト中央研究所

台湾の患者から分離されたNCKUH-21株に特異的なプロファージを発見
https://www.ncbi.nlm.nih.gov/pubmed/29213333
Gut Pathog. 2017 Nov 29;9:70. doi: 10.1186/s13099-017-0219-4. eCollection 2017.
Comparative genomic analysis of Clostridium difficile ribotype 027 strains including the newly sequenced strain NCKUH-21 isolated from a patient in Taiwan.
Suzuki H1,2, Tomita M1,2, Tsai PJ3, Ko WC4, Hung YP5,6, Huang IH7, Chen JW7.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5708112/#MOESM1
[Additional file 1. Materials and methods.(43K, docx)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5708112/bin/13099_2017_219_MOESM1_ESM.docx)

https://www.newsweekjapan.jp/stories/world/2018/04/ni150.php
うんちを寄付して報酬が年150万円
健康な人のうんちに含まれる腸内細菌を患者の腸に注入することで腸内環境を整える「便移植療法」
「クロストリジウム・ディフィシル感染症」（CDI）

https://www.newsweekjapan.jp/stories/world/2017/12/post-9194.php
「うんちカプセル」のすご過ぎる効果

2018/02/13
https://gunosy.com/articles/ao7SQ
「トレハロース問題」の真相、「感染症の原因に」論文は矛盾だらけ ネットに氾濫する不十分情報に注意を

2018年2月13日
https://lab.mykinso.com/news/180213_1/
トレハロースで増えるクロストリジウム・ディフィシル腸炎強毒性株

2018/01/24
https://www.hayashibara.co.jp/data/1416/press_tp/
雑誌Natureに掲載されたトレハロースに関する論文について｜株式会社 林原

2018年1月7日
http://aasj.jp/news/watch/7897
１月７日：食物中のトレハロースがクロストリジウム強毒株進化を促進した（Natureオンライン版掲載論文） | AASJホームページ

2017.12.25
https://medley.life/news/5a3c53f7af9647751ea32743/
プロバイオティクスはC. ディフィシル関連下痢症を予防できるか？

2017年12月25日
http://www.newsweekjapan.jp/stories/world/2017/12/post-9194.php
「うんちカプセル」のすご過ぎる効果

2017/02/03
http://www.carenet.com/news/general/hdn/43324
腸感染症に対する「糞便移植」の効果に疑問

2016年9月29日
http://www.natureasia.com/ja-jp/nature/pr-highlights/11232
微生物学：クロストリジウム・ディフィシル感染症の治療標的と考えられる受容体が新たに判明 | Nature

2016年4月7日 
http://www.natureasia.com/ja-jp/nrdp/primer/74132
クロストリジウム・ディフィシル感染症
Clostridium difficile infection

http://journal.kyorin.co.jp/journal/jscm/detail_j.php?-DB=jscm&-recid=981&-action=browse
糞便中Clostridium difficile毒素産生関連遺伝子検出キットVerigene CDFパネルの臨床的性能評価
臨床微生物：27（3），188─194，2017

http://www.chemotherapy.or.jp/guideline/index.html
特定感染症サーベイランス
「Clostridium difficile 感染症“1日”多施設共同研究」
日本化学療法学会雑誌 Vol. 65, 2017年1号（1月）　p.1～3

2016/12/24
https://www.nikkei.com/article/DGXMZO10934990R21C16A2000000/
腸内細菌は宝の山　薬剤耐性菌に効く新薬も　　：日本経済新聞
薬剤耐性菌、再発性クロストリジウム・ディフィシルの治療で、健康な人から採取した便を生理食塩水に溶解して、抗菌剤とともに患者に投与

平成28年2月8日(月)
https://www.alic.go.jp/content/000122381.pdf
腸内フローラと食品・乳製品
ディフィシル

2015.10.12
https://www.cnn.co.jp/fringe/35071695.html
ふん便が薬に、新治療「ふん便移植」が脚光　９割で症状改善 

http://terra58.hatenablog.com/entry/2015/01/04/134521
ディフィシル菌（Clostridium difficile） - てら５８



2015/3/13
http://medical.nikkeibp.co.jp/leaf/mem/pub/hotnews/nejm/201503/541189.html
C.ディフィシル感染症、米で年間45万人が発症：日経メディカル

2015年1月8日
http://www.natureasia.com/ja-jp/nature/highlights/59738
感染症：	腸内細菌がClostridium difficile耐性をもたらす

2014年11月14日
http://first.lifesciencedb.jp/archives/9479
インターロイキン22は補体系を増強することにより腸管上皮の傷害から侵入する細菌に対し抵抗性をあたえる : ライフサイエンス 新着論文レビュー

http://www.micks.jp/jhi/2014/12/20141206.html
JHIサマリー日本語版: クロストリジウム・ディフィシル（Clostridium difficile）リボタイプ 027 は長期ケア施設から転院した入院患者での保有率が高い

http://www.micks.jp/jhi/2010/08/20100808.html
JHIサマリー日本語版: クロストリジウム・ディフィシル（Clostridium difficile）リボタイプ078による感染：アイルランド共和国における初の症例報告

2013/10/10
http://www.carenet.com/news/journal/carenet/36333
ゲノム解析で判明、C. difficile感染の伝播経路は複数存在／NEJM

2013年4月15日
http://www.city.yokohama.lg.jp/kenko/eiken/idsc/disease/clostridium1.html
横浜市衛生研究所：クロストリジウム-ディフィシル感染症について

2012年12月10日
http://www.natureasia.com/ja-jp/ng/pr-highlights/1996
医療に関連したClostridium difficileの流行 | Nature Genetics

2010年10月
http://www.eiken.co.jp/modern_media/backnumber/pdf/MM1010_01.pdf
ディフィシル菌感染症の基礎と臨床

2006年7月1日
http://www.natureasia.com/ja-jp/ng/38/7/ng1830/
クロストリジウム-ディフィシレ菌のゲノム | Nature Genetics

2005年11月
http://www.bdj.co.jp/safety/articles/ignazzo/1f3pro00000sj5kj.html
I's eye： クロストリジウム・ディフィシルの真実 | 日本BD

----------

## Geobacter

https://ja.wikipedia.org/wiki/ゲオバクター属

2016-2017
http://www.u-shizuoka-ken.ac.jp/file/88sekikawa.pdf
主な異化的鉄還元細菌としてGeobacter属細菌(ジオバクター)とShewanella属細菌(シュ ワネラ)が挙げられます。

https://mainichi.jp/articles/20160604/ddl/k22/040/140000c
微生物発電：１００倍超に　二又・静岡大教授が効率化物質を発見　／静岡 - 毎日新聞

2015年06月09日
http://eetimes.jp/ee/articles/1506/09/news041.html
東大駒場リサーチキャンパス公開2015：赤い微生物「ジオバクター」の燃料電池、廃水処理システムを変える？ - EE Times Japan

Jan. 23, 2015
https://www.jst.go.jp/pr/announce/20120605/index.html
微生物が互いに電子をやり取りする未知の「電気共生」を発見

- 比較ゲノム
https://www.hindawi.com/journals/isrn/2013/850179/
A Whole Genome Pairwise Comparative and Functional Analysis of Geobacter sulfurreducens PCA

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/22967216
BMC Genomics. 2012 Sep 12;13:471. doi: 10.1186/1471-2164-13-471.
Comparative genomic analysis of Geobacter sulfurreducens KN400, a strain with enhanced capacity for extracellular electron transfer and electricity production.
Butler JE1, Young ND, Aklujkar M, Lovley DR.

5/24/12
https://io9.gizmodo.com/5908318/10-surprising-things-that-bacteria-like-to-eat
Frank Miller Is Illustrating a YA Novel That Will Tell the Lady of the Lake's Origin
10. Nuclear Waste
Geobacter sulfurreducens
9. Arsenic

酢酸を電子供与体に、硝酸を電子受容体として、土壌細菌の代表株であるゲオバクター（Geobacter sulfurreducens ）とチオバチルス（Thiobacillus denitrificans ）を共培養

Mar 28, 2012
https://www.youtube.com/watch?v=LxaAWbN2koo
Michigan State University 2011 President's Report: Zapping nuclear waste. - YouTube

2011年9月7日
http://sustainablejapan.net/?p=452
発電しながら放射能汚染土壌を浄化する微生物「ジオバクター」の謎、ミシガン州立大が解明 | SJN News　再生可能エネルギー最新情報

2011
http://www.jseb.jp/jeb/11-0102/11-12-033.pdf
微生物の細胞外電子伝達鉄還元細菌の細胞外電子伝達についての研究は She- wanella oneidensis や Geobacter sulfurreducens を中心に 先駆的な研究がなされてきた。

https://wired.jp/2011/09/16/ウランの地下水汚染を防ぐ細菌、そのメカニズム/

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/20078895
BMC Genomics. 2010 Jan 17;11:40. doi: 10.1186/1471-2164-11-40.
Evolution of electron transfer out of the cell: comparative genomics of six Geobacter genomes.
Butler JE1, Young ND, Lovley DR.

https://www.ncbi.nlm.nih.gov/pubmed/18844997
BMC Genomics. 2008 Oct 9;9:471. doi: 10.1186/1471-2164-9-471.
Comparative genomics of Geobacter chemotaxis genes reveals diverse signaling function.
Tran HT1, Krushkal J, Antommattei FM, Lovley DR, Weis RM.

----------

## Pseudomonas
- https://ja.wikipedia.org/wiki/シュードモナス属
- https://en.wikipedia.org/wiki/Pseudomonas

- review

https://www.ncbi.nlm.nih.gov/pubmed/21361996
FEMS Microbiol Rev. 2011 Jul;35(4):652-80. doi: 10.1111/j.1574-6976.2011.00269.x. Epub 2011 Mar 25.
Pseudomonas genomes: diverse and adaptable.
Silby MW1, Winstanley C, Godfrey SA, Levy SB, Jackson RW.

Members of the genus Pseudomonas inhabit a wide variety of environments, which is reflected in their versatile metabolic capacity and broad potential for adaptation to fluctuating environmental conditions. 

https://onlinelibrary.wiley.com/doi/full/10.1111/j.1574-6976.2011.00269.x
Bioremediation 
Pseudomonas stutzeri– a nitrogen‐fixing nonfluorescent pseudomonad

- 分類
https://www.ncbi.nlm.nih.gov/pubmed/21811795
Microb Ecol. 2012 Feb;63(2):239-48. doi: 10.1007/s00248-011-9914-8. Epub 2011 Aug 3.
Defining the Pseudomonas genus: where do we draw the line with Azotobacter?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3275731/
 [59]. 
Owing to the low resolution in 16S rRNA sequences for these genomes, they conducted a phylogenetic analysis of 25 protein-coding genes, some of which were housekeeping genes.

Although they are clustered together in the 16S rRNA tree, the low resolution of the 16S rRNA phylogenetic analysis has been noted by others [39, 59]. It does show, however, that they have a common ancestral rrs gene that is closer to each other than to other species. 

----------
### Pseudomonas aeruginosa

https://www.ncbi.nlm.nih.gov/pubmed/28606056
BMC Genomics. 2017 Jun 12;18(1):459. doi: 10.1186/s12864-017-3842-z.
Comparative genome and transcriptome analysis reveals distinctive surface characteristics and unique physiological potentials of Pseudomonas aeruginosa ATCC 27853.
Cao H1, Lai Y2,3, Bougouffa S4, Xu Z2, Yan A5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5467263/
Prophages

https://www.ncbi.nlm.nih.gov/pubmed/25961859
PLoS One. 2015 May 11;10(5):e0126468. doi: 10.1371/journal.pone.0126468. eCollection 2015.
What It Takes to Be a Pseudomonas aeruginosa? The Core Genome of the Opportunistic Pathogen Updated.

https://www.ncbi.nlm.nih.gov/pubmed/25168460
BMC Genomics. 2014 Aug 29;15:737. doi: 10.1186/1471-2164-15-737.
Characterization of the core and accessory genomes of Pseudomonas aeruginosa using bioinformatic tools Spine and AGEnt.

https://www.ncbi.nlm.nih.gov/pubmed/21119020
Microbiol Mol Biol Rev. 2010 Dec;74(4):621-41. doi: 10.1128/MMBR.00027-10.
The accessory genome of Pseudomonas aeruginosa.
Kung VL1, Ozer EA, Hauser AR.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3008168/
The majority of the P. aeruginosa accessory genome can be grouped into four broad categories: (i) integrative and conjugative elements (ICEs), (ii) replacement islands, (iii) prophages and phage-like elements, and (iv) transposons, insertion sequences (ISs), and integrons.

secretion system

- コドン使用
https://www.ncbi.nlm.nih.gov/pubmed/12036591
Gene. 2002 May 1;289(1-2):131-9.
Synonymous codon usage in Pseudomonas aeruginosa PA01.
Grocock RJ1, Sharp PM.

----------
### Pseudomonas stutzeri
https://microbewiki.kenyon.edu/index.php/Pseudomonas_stutzeri

https://ja.wikipedia.org/wiki/シュードモナス・スタッツェリ
P. stutzeriは、環境中から同種および異種のプラスミドDNAを取り込み、自身のゲノムに統合する自然形質転換[ 英: natural transformation ]をすることができる。この能力は多様な環境に適応することを可能にし、P. stutzeriは世界中の幅広い環境中から見出せる。

http://www.jarmam.gr.jp/situmon3/pseudomonas-stutzeri.html
Pseudomonas stutzeriの病原性と分布

https://www.ncbi.nlm.nih.gov/pubmed/16760312
Microbiol Mol Biol Rev. 2006 Jun;70(2):510-47.
Biology of Pseudomonas stutzeri.
Lalucat J1, Bennasar A, Bosch R, García-Valdés E, Palleroni NJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489536/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489536/table/t1/
TABLE 1.
P. stutzeri strains cited in the text, with relevant characteristics, origins, and references

19SMN4	DSM6084	Ref, gv 4	1988	Marine sediment; naphthalene degrader; Barcelona, Spain

JM300	DSM10701	Ref, gv 8	Pre-1980	Soil; California; natural transformation model organism

CMT.9.A	DSM4166		1987	Sorghum nutans rhizosphere; nitrogen fixer; Germany	189

A15	LMG10652	gv 1	1981	Rice paddy; nitrogen fixer; Southeast Asia; formerly Alcaligenes faecalis

JM300	DSM10701	Ref, gv 8	Pre-1980	Soil; California; natural transformation model organism	60

SP1402	DSM6083	Former Ref, gv 6; P. balearica	1988	Wastewater; 2-methylnaphthalene degrader; Mallorca, Spain


- NYC地下鉄
https://www.ncbi.nlm.nih.gov/pubmed/26594662
Cell Syst. 2015 Jul 29;1(1):72-87. Epub 2015 Mar 3.
Geospatial Resolution of Human and Bacterial Diversity with City-Scale Metagenomics.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4651444/
The most prevalent bacterial species on the subway was Pseudomonas stutzeri, with enrichment in lower Manhattan (Figure 1E),

https://github.com/haruosuz/metasub/blob/master/references/README.md#new-york-subway-microbiome

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/28649138
Nat Rev Microbiol. 2017 Sep;15(9):517-530. doi: 10.1038/nrmicro.2017.61. Epub 2017 Jun 26.
Pseudomonas predators: understanding and exploiting phage-host interactions.
De Smet J1, Hendrix H1, Blasdel BG1, Danis-Wlodarczyk K1, Lavigne R1.
The Pseudomonas genus is one of the most ecologically important groups of bacteria and includes species that are plant commensals (Pseudomonas stutzeri and Pseudomonas fluorescens) and 

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/26025640
Res Microbiol. 2015 Nov;166(9):668-76. doi: 10.1016/j.resmic.2015.05.001. Epub 2015 May 27.
Identification and genomic analysis of temperate Pseudomonas bacteriophage PstS-1 from the Japan trench at a depth of 7000 m.
Yoshida M1, Yoshida-Takashima Y2, Nunoura T2, Takai K2.
A temperate bacteriophage (termed vB_PstS-1) was identified from the psychrotolerant gammaproteobacterium Pseudomonas stutzeri 1-1-1b, 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/27233093
Environ Microbiol. 2016 Oct;18(10):3535-3549. doi: 10.1111/1462-2920.13390. Epub 2016 Jul 7.
Comparative genomic, proteomic and exoproteomic analyses of three Pseudomonas strains reveals novel insights into the phosphorus scavenging capabilities of soil bacteria.
Lidbury ID1, Murphy AR2, Scanlan DJ2, Bending GD2, Jones AM2, Moore JD3, Goodall A4, Hammond JP4,5, Wellington EM2.
Pseudomonas stutzeri DSM4166
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5082522/
Table 1
Comparative genomic analysis of selected proteins involved in the recycling of P among Pseudomonas isolates.
P. stutzeri A1501	P. stutzeri DSM4166	P. stutzeri DSM10701

- VI型分泌装置
https://www.ncbi.nlm.nih.gov/pubmed/27560648
Genet Mol Biol. 2016 Jul-Sep;39(3):465-73. doi: 10.1590/1678-4685-GMB-2015-0186. Epub 2016 Jul 7.
Comparative genomics of an endophytic Pseudomonas putida isolated from mango orchard.
Asif H1, Studholme DJ2, Khan A1, Aurongzeb M1, Khan IA1, Azim MK1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5004826/
TypeVI secretion system (T6SS) in Pseudomonas putida TJ151

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/26519390
Appl Environ Microbiol. 2015 Oct 30;82(1):375-83. doi: 10.1128/AEM.02612-15. Print 2016 Jan 1.
Diversity of Pseudomonas Genomes, Including Populus-Associated Isolates, as Revealed by Comparative Genome Analysis.
Jun SR1, Wassenaar TM2, Nookaew I3, Hauser L3, Wanchai V3, Land M3, Timm CM3, Lu TY3, Schadt CW3, Doktycz MJ3, Pelletier DA3, Ussery DW4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702629/
(iv) The Azotobacter genome is found in the P. stutzeri group. The A. vinelandii DJ genome was included in this analysis because previous findings, based on comparative genomics of Azotobacter and Pseudomonas (25), showed that it was closely related. Its position is indicated by arrow B in Fig. 1 and is consistent with inclusion in the P. stutzeri group. 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/25898829
BMC Genomics. 2015 Apr 18;16:313. doi: 10.1186/s12864-015-1488-2.
Pan-genome dynamics of Pseudomonas gene complements enriched across hexachlorocyclohexane dumpsite.
Sharma A1, Sangwan N2, Negi V3, Kohli P4, Khurana JP5, Rao DL6, Lal R7.
Pseudomonas sp. strain RL
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4405911/
largely divergent P. stutzeri group (Figure 1A). 
However, two sub-groups were identified for P. stutzeri clade, first with P. stutzeri CCUG 29243 and ATCC 14405, and second containing P. stutzeri A1501, T13 and XLDN-R (Figure 1A). 

- 比較ゲノム、可動性遺伝因子
https://www.ncbi.nlm.nih.gov/pubmed/24803113
Environ Microbiol. 2015 Jan;17(1):91-104. doi: 10.1111/1462-2920.12498. Epub 2014 May 28.
Comparative genome analysis of Pseudomonas knackmussii B13, the first bacterium known to degrade chloroaromatic compounds.
The B13 genome contains at least eight genomic islands [prophages and integrative conjugative elements (ICEs)], which were absent in closely related pseudomonads. 
https://onlinelibrary.wiley.com/doi/pdf/10.1111/1462-2920.12498
P. stutzeri DSM 4166

- ファージ
https://www.ncbi.nlm.nih.gov/pubmed/22923799
J Virol. 2012 Sep;86(18):10239. doi: 10.1128/JVI.01605-12.
Genome sequence of the broad-host-range Pseudomonas phage Φ-S1.
Sillankorva S1, Kropinski AM, Azeredo J.
The lytic phage Φ-S1 was isolated from the Greater Vancouver Sewage Processing plant using Pseudomonas fluorescens PW. This virus lysed most of the fluorescent Pseudomonas species tested and one strain of Pseudomonas stutzeri. 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/27922098
Sci Rep. 2016 Dec 6;6:38699. doi: 10.1038/srep38699.
Comparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.
Koehorst JJ1, van Dam JC1, van Heck RG1, Saccenti E1, Dos Santos VA1,2, Suarez-Diez M1, Schaap PJ1.
http://www.nature.com/articles/srep38699
- P. aeruginosa and P. stutzeri clades are conserved while P. putida and P. fluorescence clades shows the addition of different species.
- Figure 4: Domain based distance tree of 58 Pseudomonas strains.
- The phylogenetic clusters corresponding to the most abundant species (P. stutzeri, P. mendocina, P. aeruginosa and P. putida) are colour-shadowed.

- 亜ヒ酸酸化細菌 Pseudomonas stutzeri TS44 のドラフトゲノム配列
https://www.ncbi.nlm.nih.gov/pubmed/22843599
J Bacteriol. 2012 Aug;194(16):4473-4. doi: 10.1128/JB.00907-12.
Genome sequence of the moderately halotolerant, arsenite-oxidizing bacterium Pseudomonas stutzeri TS44.
Li X1, Gong J, Hu Y, Cai L, Johnstone L, Grass G, Rensing C, Wang G.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3416245/
The genome contains at least 20 genes encoding functions related to arsenite oxidation and resistance, mainly located on contig 00030 (AJXE01000030; ars operon, arsC1-arsR-arsC2-ACR3-arsH-DSP-GAPDH-MFS, and an adjacent ars-aox operon, arsD-arsA-aioA-aioB) and contig 00031 (AJXE01000031; ars operon, ACR3-arsR-arsH-GAPDH-MFS) (9, 11, 13). Notably, aioAB (encoding arsenite oxidase) was absent in the other five P. stutzeri genomes, suggesting recent acquisition of aioAB by strain TS44 via horizontal gene transfer. However, a two-component system, aioS aioR, involved in regulating the expression of aioAB, was not identified in the genome, indicating a potentially novel regulatory mechanism (10, 14). In addition, numerous genes responsible for resistance to other metals (copper, mercury, chromate, cadmium, and zinc) were also identified.

https://www.sciencedirect.com/science/article/pii/S0964830516306515
August 2017
In silico approach for bioremediation of arsenic by structure prediction and docking studies of arsenite oxidase from Pseudomonas stutzeri TS44

- ヒ素蓄積
https://www.ncbi.nlm.nih.gov/pubmed/19568912
Environ Health Prev Med. 2008 Sep;13(5):257-63. doi: 10.1007/s12199-008-0038-9. Epub 2008 Jul 4.
Arsenic accumulation by Pseudomonas stutzeri and its response to some thiol chelators.
Joshi DN1, Patel JS, Flora SJ, Kalia K.

#### Pseudomonas stutzeri genomes

##### P.stutzeri_group

- Pseudomonas mendocina ymp

https://www.ncbi.nlm.nih.gov/nuccore/CP000680
Pseudomonas mendocina ymp, complete genome

https://www.ncbi.nlm.nih.gov/bioproject/17457
Pseudomonas mendocina ymp (ID 17457) - BioProject - NCBI

https://www.ncbi.nlm.nih.gov/bioproject/?term=Pseudomonas%20mendocina%20ymp
A bacterium which is able to degrade pentachlorophenol

https://genome.jgi.doe.gov/portal/pseme/pseme.home.html
Strain ymp was isolated as part of the Yucca Mountain Project from sediment in a surface holding pond of a drilling operation at the Nevada Test Site, NV (the proposed site of the  high-level nuclear repository).

http://bacmap.wishartlab.com/organisms/495
P.mendocina strain ymp, a strict aerobe, is a pentachlorophenol (PCP)-degrading bacterium isolated from PCP-contaminated soil.
Isolation site	sediment, surface holding pond, Nevada Test Site, NV
Isolation country	USA

- LH-42

https://www.ncbi.nlm.nih.gov/pubmed/28684578
Genome Announc. 2017 Jul 6;5(27). pii: e00589-17. doi: 10.1128/genomeA.00589-17.
Draft Genome Sequence of Pseudomonas stutzeri LH-42, Isolated from Petroleum-Contaminated Soil.
Zhang M1, Ma L1, Yang Y2, Hu T1, Liu X1, Zhang X1, Hua Y1, Gao Y1, Zhu Z1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5502859/

- KOS6

https://www.ncbi.nlm.nih.gov/pubmed/23405317
Genome Announc. 2013 Jan;1(1). pii: e00072-12. doi: 10.1128/genomeA.00072-12. Epub 2013 Jan 31.
Draft Genome of the Nitrogen-Fixing Bacterium Pseudomonas stutzeri Strain KOS6 Isolated from Industrial Hydrocarbon Sludge.
Grigoryeva TV1, Laikov AV, Naumova RP, Manolov AI, Larin AK, Karpova IY, Semashko TA, Alexeev DG, Kostryukova ES, Müller R, Govorun VM.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3569306/
Strain KOS6 was most closely related to strain CCUG 29243 (blast total score, 4.560e+06, 70% query coverage), 

- Pseudomonas balearica DSM 6083T

https://www.ncbi.nlm.nih.gov/pubmed/27103708
Genome Announc. 2016 Apr 21;4(2). pii: e00217-16. doi: 10.1128/genomeA.00217-16.
Complete Genome Sequence of Pseudomonas balearica DSM 6083T.
Bennasar-Figueras A1, Salvà-Serra F2, Jaén-Luchoro D3, Seguí C3, Aliaga F4, Busquets A3, Gomila M3, Moore ER5, Lalucat J3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4841123/

##### P.stutzeri_1

- RCH2

https://www.ncbi.nlm.nih.gov/nuccore?term=60029%5BBioProject%5D

https://www.ncbi.nlm.nih.gov/pubmed/28194258
Stand Genomic Sci. 2017 Feb 8;12:23. doi: 10.1186/s40793-017-0233-7. eCollection 2017.
Complete genome sequence of Pseudomonas stutzeri strain RCH2 isolated from a Hexavalent Chromium [Cr(VI)] contaminated site.
Chakraborty R1, Woo H2, Dehal P1, Walker R1, Zemla M1, Auer M1, Goodwin LA3, Kazakov A1, Novichkov P1, Arkin AP1, Hazen TC1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299692/
and includes 3 circular plasmids of 12,763 bp, 9,865 bp and 2,804 bp for a total genome size of 4,600,489 bp. 
No orthologs for plasmid genes of RCH2 were found in five other P. stutzeri strains. 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299692/table/Tab1/?report=objectonly
MIGS-6	Habitat	Cr(VI) contaminated aquifer	IDA

- 19SMN4

https://www.ncbi.nlm.nih.gov/nuccore/CP007509
Pseudomonas stutzeri strain 19SMN4 , complete genomes

https://www.ncbi.nlm.nih.gov/nuccore/CP007510
Pseudomonas stutzeri strain 19SMN4 plasmid pLIB119, complete plasmid

https://www.ncbi.nlm.nih.gov/bioproject/PRJNA242326
Pseudomonas stutzeri strain:19SMN4 (ID 242326) - BioProject - NCBI

https://www.ncbi.nlm.nih.gov/pubmed/8161187
Appl Environ Microbiol. 1994 Mar;60(3):966-72.
Comparative biochemical and genetic analysis of naphthalene degradation among Pseudomonas stutzeri strains.
Rosselló-Mora RA1, Lalucat J, García-Valdés E.

- CCUG 29243

https://www.ncbi.nlm.nih.gov/nuccore/CP003677
Pseudomonas stutzeri CCUG 29243, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/23144395
J Bacteriol. 2012 Dec;194(23):6642-3. doi: 10.1128/JB.01753-12.
Complete genome sequence of the naphthalene-degrading bacterium Pseudomonas stutzeri AN10 (CCUG 29243).
Brunet-Galmés I1, Busquets A, Peña A, Gomila M, Nogales B, García-Valdés E, Lalucat J, Bennasar A, Bosch R.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3497477/
Finally, a putative prophage sequence (42.7 kb in length) was also detected within the 400-kb region (coordinates 1,476,863 to 1,524,049). All these data suggest that this region might be a specific area of the chromosome used to stabilize foreign genetic material. In relation to this, as observed in other P. stutzeri strains (4, 16), the complete repertoire of genes needed for the acquisition of foreign DNA by natural transformation (pilA, pilBCD, pilTU, exbB, and comA) has been detected.

##### P.stutzeri_2

- SLG510A3-8

https://www.ncbi.nlm.nih.gov/nuccore/CP011854
Pseudomonas stutzeri strain SLG510A3-8, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/26144046
J Biotechnol. 2015 Oct 10;211:1-2. doi: 10.1016/j.jbiotec.2015.06.421. Epub 2015 Jul 2.
Complete genome sequence of the petroleum-emulsifying bacterium Pseudomonas stutzeri SLG510A3-8.
Hu B1, Nie Y1, Geng S1, Wu XL2.
https://linkinghub.elsevier.com/retrieve/pii/S0168-1656(15)30055-9

- DSM 4166

https://www.ncbi.nlm.nih.gov/nuccore/CP002622
Pseudomonas stutzeri DSM 4166, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/21515765
J Bacteriol. 2011 Jul;193(13):3422-3. doi: 10.1128/JB.05039-11. Epub 2011 Apr 22.
Complete genome sequence of the nitrogen-fixing and rhizosphere-associated bacterium Pseudomonas stutzeri strain DSM4166.
Yu H1, Yuan M, Lu W, Yang J, Dai S, Li Q, Yang Z, Dong J, Sun L, Deng Z, Zhang W, Chen M, Ping S, Han Y, Zhan Y, Yan Y, Jin Q, Lin M.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3133286/

- CGMCC 1.1803

https://www.ncbi.nlm.nih.gov/nuccore/CP002881
Pseudomonas stutzeri strain CGMCC 1.1803, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/21994926
J Bacteriol. 2011 Nov;193(21):6095. doi: 10.1128/JB.06061-11.
Complete genome sequence of the type strain Pseudomonas stutzeri CGMCC 1.1803.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3194927/
, and no plasmid was detected. 

-  A1501

https://www.ncbi.nlm.nih.gov/nuccore/CP000304
Pseudomonas stutzeri A1501, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/18495935
Proc Natl Acad Sci U S A. 2008 May 27;105(21):7564-9. doi: 10.1073/pnas.0801093105. Epub 2008 May 21.
Nitrogen fixation island and rhizosphere competence traits in the genome of root-associated Pseudomonas stutzeri A1501.
Yan Y1, Yang J, Dou Y, Chen M, Ping S, Peng J, Lu W, Zhang W, Yao Z, Li H, Liu W, He S, Geng L, Zhang X, Yang F, Yu H, Zhan Y, Li D, Lin Z, Wang Y, Elmerich C, Lin M, Jin Q.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2396677/

##### P.stutzeri_3

- 28a24

https://www.ncbi.nlm.nih.gov/nuccore/CP007441
Pseudomonas stutzeri strain 28a24, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/24903873
Genome Announc. 2014 Jun 5;2(3). pii: e00543-14. doi: 10.1128/genomeA.00543-14.
Complete Genome Sequence of the Highly Transformable Pseudomonas stutzeri Strain 28a24.
Smith BA1, Dougherty KM1, Baltrus DA2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047452/
USA

- 273

https://www.ncbi.nlm.nih.gov/nuccore/CP015641
Pseudomonas stutzeri strain 273, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/29675016
Front Microbiol. 2018 Apr 5;9:682. doi: 10.3389/fmicb.2018.00682. eCollection 2018.
Genetic and Physiological Adaptations of Marine Bacterium Pseudomonas stutzeri 273 to Mercury Stress.
Zheng R1,2,3, Wu S4, Ma N1,2,3, Sun C1,2.

https://www.ncbi.nlm.nih.gov/pubmed/28698510
Mar Drugs. 2017 Jul 10;15(7). pii: E218. doi: 10.3390/md15070218.
Genome Sequence of Pseudomonas stutzeri 273 and Identification of the Exopolysaccharide EPS273 Biosynthesis Locus.
Wu S1, Zheng R2,3,4, Sha Z5, Sun C6,7.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5532660/
In our previous study, P. stutzeri 273 was isolated from the sediments of the East China Sea and exhibited great potential in combating biofilm-related infection and biofouling in marine aquaculture [13].

https://www.ncbi.nlm.nih.gov/pubmed/26903981
Front Microbiol. 2016 Feb 8;7:102. doi: 10.3389/fmicb.2016.00102. eCollection 2016.
Antibiofilm and Anti-Infection of a Marine Bacterial Exopolysaccharide Against Pseudomonas aeruginosa.
Wu S1, Liu G2, Jin W3, Xiu P2, Sun C3.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744842/
The marine bacteria strains used in this study were isolated from the sediments of East China Sea


- DSM 10701

https://www.ncbi.nlm.nih.gov/nuccore/CP003725
Pseudomonas stutzeri DSM 10701, complete genome

https://www.ncbi.nlm.nih.gov/pubmed/22965097
J Bacteriol. 2012 Oct;194(19):5477-8. doi: 10.1128/JB.01257-12.
Genome sequence of Pseudomonas stutzeri strain JM300 (DSM 10701), a soil isolate and model organism for natural transformation.
Busquets A1, Peña A, Gomila M, Bosch R, Nogales B, García-Valdés E, Lalucat J, Bennasar A.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3457215/
Genes coding for products necessary for natural transformation ability were found (type IV pili, pilAI, pilC, pilT, and pilU, comA, and exbB). No nitrogen fixation genes (nif) or extrachromosomal elements were found. Predicted phage-related sequences, transposons, and insertion elements were detected.

#### P.stutzeri_frontier

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2018.01458/full
Front. Microbiol., 02 July 2018 | https://doi.org/10.3389/fmicb.2018.01458
Frontiers | Comparative Genomics of Pseudomonas sp. Strain SI-3 Associated With Macroalga Ulva prolifera, the Causative Species for Green Tide in the Yellow Sea | Microbiology

Keywords: Pseudomonas, genome sequence, Comparative genomics, Ulva prolifera, algae-associated bacteria

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2018.00682/full
Front. Microbiol., 05 April 2018 | https://doi.org/10.3389/fmicb.2018.00682
Frontiers | Genetic and Physiological Adaptations of Marine Bacterium Pseudomonas stutzeri 273 to Mercury Stress | Microbiology

Keywords: marine, Pseudomonas stutzeri, mercury, stress, motility, flagella

- 
https://www.frontiersin.org/articles/10.3389/fmicb.2015.00214/full
Front. Microbiol., 18 March 2015 | https://doi.org/10.3389/fmicb.2015.00214
Phylogenomics and systematics in Pseudomonas

Keywords: Pseudomonas, genomics, multilocus sequence analysis, taxonomy, systematics

#### P.stutzeri_denitrification

https://www.ncbi.nlm.nih.gov/pubmed/16483137
Sci China C Life Sci. 2005 Dec;48(6):585-92.
Structural and functional analysis of denitrification genes in Pseudomonas stutzeri A1501.
Yan Y1, Yang J, Chen L, Yang F, Dong J, Xue Y, Xu X, Zhu Y, Yao Z, Lin M, Wang Y, Jin Q.
Four gene clusters associated with denitrification were identified in the genome of A1501 strain, nar, nir, nor and nos, including 40 genes totally, which encode proteins for substance transportation, gene regulation and reductases. The three gene clusters, nir, nor and nos are adjacent on chromosome and are far from nar gene cluster. 

https://www.ncbi.nlm.nih.gov/pubmed/12618453
J Bacteriol. 2003 Mar;185(6):1895-902.
Operon structure and regulation of the nos gene region of Pseudomonas stutzeri, encoding an ABC-Type ATPase for maturation of nitrous oxide reductase.
Honisch U1, Zumft WG.

https://www.ncbi.nlm.nih.gov/pubmed/12576577
Microbiology. 2003 Jan;149(Pt 1):29-36.
Transcriptional regulation of the nos genes for nitrous oxide reductase in Pseudomonas aeruginosa.
Arai H1, Mizutani M, Igarashi Y.
The genes for nitrous oxide (N(2)O) reduction, nosRZDFYL, are clustered on the chromosome of Pseudomonas aeruginosa. 
http://mic.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.25936-0#tab2 
Nitrate (NO−3) is reduced through nitrite (NO−2), nitric oxide (NO) and nitrous oxide (N 2O) to dinitrogen (N 2) by denitrifying bacteria. N 2O reductase (N 2OR) catalyses the last step of the denitrification pathway, the two-electron reduction of N 2O to N 2. N 2OR has been purified and characterized from many bacteria, such as Pseudomonas stutzeri ( Coyle et al., 1985 ), 

The organization and regulation of the N 2OR genes have been investigated intensively in P. stutzeri. In this bacterium, the genes for N 2O reduction, nosRZDFYL, are clustered with the genes for other denitrification enzymes, NO−2 reductase (NIR) and NO reductase (NOR) ( R9 Braun & Zumft, 1992; Jüngst et al., 1991 ). The structural gene for N 2OR is nosZ ( Viebrock & Zumft, 1988 ), the nosDFY gene products are thought to be involved in the processing and insertion of copper into N 2OR ( Zumft et al., 1990 ), the nosL gene product is proposed to be an outer membrane disulfide isomerase ( Dreusch et al., 1996 ) and nosR encodes a putative membrane protein ( Cuypers et al., 1992 ). The nosR gene product is proposed to be an N 2O-sensing regulator that is required for the expression of the N 2OR genes ( Vollack & Zumft, 2001 ). The nos gene cluster of P. stutzeri is transcribed as at least three operons from promoters located upstream of nosR, nosZ and nosD ( Cuypers et al., 1995 ).

Genome sequencing of P. aeruginosa PAO1 has shown that the arrangement of the nosRZDFYL genes (PA3391–PA3396) is identical to that of P. stutzeri ( Stover et al., 2000 ); 

https://www.ncbi.nlm.nih.gov/pubmed/9841665
Microbiol Mol Biol Rev. 1998 Dec;62(4):1046-78.
Molecular genetics of the genus Paracoccus: metabolically versatile bacteria with bioenergetic flexibility.
Baker SC1, Ferguson SJ, Ludwig B, Page MD, Richter OM, van Spanning RJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC98939/
Organization of denitrification genes.
In P. denitrificans, nir genes (associated with nitrite reduction) and nor genes (associated with nitric oxide reduction) are clustered on a 17.7-kbp fragment, essentially transcribed divergently from one another (Fig. ​(Fig.4).4).
In Pseudomonas stutzeri, the other organism for which the genetics of denitrification have been extensively studied, the nos genes (nitrous oxide reductase associated) are also found in the same 30-kbp region of the genome as the nir and nor genes (34, 97, 347, 348, 350). The order of the genes within the nir, nor, and nos clusters varies among organisms, but the structural genes (nirS and norCB) are transcribed from the 5′ end of an operon (34, 59, 252). 

FIG. 4
Denitrification gene clusters of Pseudomonas stutzeri (A), Pseudomonas aeruginosa (B), and P. denitrificans Pd1222 (C). Reproduced with permission from reference 251a.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC98939/figure/F4/

#### P.stutzeri_hydrocarbon

http://www.bio.nite.go.jp/mifup/wiki/index.php?title=芳香族炭化水素分解

https://www.ncbi.nlm.nih.gov/pubmed/29217799
Genome Announc. 2017 Dec 7;5(49). pii: e01373-17. doi: 10.1128/genomeA.01373-17.
Draft Genome Sequence of Pseudomonas stutzeri Strain 19, an Isolate Capable of Efficient Degradation of Aromatic Hydrocarbons.
Brown LM1, Gunasekera TS1, Ruiz ON2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5721144/
P. stutzeri 19 was shown, through gas chromatography-mass spectrometry (GC-MS) analysis, to efficiently metabolize toluene, xylenes, and 1,2,4-trimethyl benzene.

The NCBI PGAP predicted multiple genes involved in hydrocarbon degradation, including catechol 1,2-dioxygenase, 

https://www.sciencedirect.com/science/article/pii/S1878818117300038
Biocatalysis and Agricultural Biotechnology
Volume 10, April 2017, Pages 20-29
Optimization of conditions for polycyclic aromatic hydrocarbons (PAHs) degradation by Pseudomonas stutzeri P2 isolated from Chirimiri coal mines

PCR amplification of dioxygenase genes produced fragments of 350 bp, 867 bp and 350 bp in length for PAH-RHD, catechol 2, 3-dioxygenase and catechol 1, 2-dioxygenase, respectively suggesting the representative of Pseudomonas as the significant PAHs degrader.

https://www.tandfonline.com/doi/abs/10.1080/10406638.2015.1033434
Characterization of Phenol Metabolization by P. stutzeri N2
Published online: 07 Oct 2015

https://www.ncbi.nlm.nih.gov/pubmed/8161187
Appl Environ Microbiol. 1994 Mar;60(3):966-72.
Comparative biochemical and genetic analysis of naphthalene degradation among Pseudomonas stutzeri strains.
Rosselló-Mora RA1, Lalucat J, García-Valdés E.

----------
## codon usage bias

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549432/
Variation in the strength of selected codon usage bias among bacteria

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC549432/table/tbl1/
    Pseaer	4	62	5566	67	87	74	−0.019	(0.484/−0.507)	940	AE004091	Pseudomonas aeruginosa
    Pseput	7	74	5350	62	77	64	0.917	(0.360/−0.317)	966	AE015451	Pseudomonas putida
    Psesyr	5	64	5566	58	71	58	0.701	(0.255/−0.243)	958	AE016853	Pseudomonas syringae

If the 40 genes are compared with only those on the leading strand, the S-value becomes 0.42, indicative of weak selection. Pseudomonas aeruginosa has an S-value close to zero (−0.02), providing no evidence for selection, whereas we previously found small but significant differences in codon usage between highly expressed and other genes (27). This discrepancy arises because the largest components of selected bias in this species relate to codons for Ser (especially UCC), Thr (ACC), Ala (GCU), Arg (CGU) and Gly (GGU), whereas frequencies of the C-ending codons for Phe, Tyr, Ile and Asn (used to calculate S) differ little between highly expressed genes and the genome as a whole (27).

Perhaps the most surprising example of low codon usage bias is P.aeruginosa. This species can grow quite rapidly (doubling times <1 h) in laboratory planktonic cultures and is metabolically highly versatile. It is moderately recombinogenic via plasmid transfer, and there appear to be many horizontally transferred genes in its genome (27). The low selected bias was apparent in a full analysis of codon usage in this species (27), as well as the S-value calculated here. Selected codon usage bias is rather stronger in the two other Pseudomonas species analysed (Table 1). These paradoxical observations perhaps highlight our ignorance of the evolutionary history of even ‘well-known’ bacterial species.

----------
## Rhodococcus

- https://ja.wikipedia.org/wiki/ロドコッカス属
- https://ja.wikipedia.org/wiki/ロドコッカス・エクイ
- https://en.wikipedia.org/wiki/Rhodococcus
- https://microbewiki.kenyon.edu/index.php/Rhodococcus
Bacteria; Actinobacteria; Actinobacteria (class); Actinobacteridae; Actinomycetales; Corynebacterineae; Nocardiaceae; Rhodococcus

https://twitter.com/search?q=Rhodococcus

https://twitter.com/MattPerkins3/status/950176390749589505
In vitro studies have found Rhodococcus equi to usually be susceptible to erythromycin, rifampin, ciprofloxacin, vancomycin, aminoglycosides, imipenem & meropenem. Usually resistant to most other beta-lactams, variably susceptible to clinda, ceftriaxone, TMP-SMX & tetracycline.

https://www.ncbi.nlm.nih.gov/pubmed/28369330
Genome Biol Evol. 2017 May 1;9(5):1241-1247. 
Comparative Genomics of Rhodococcus equi Virulence Plasmids Indicates Host-Driven Evolution of the vap Pathogenicity Island.
MacArthur I1, Anastasi E1, Alvarez S1, Scortti M1,2, Vázquez-Boland JA1,2,3.

https://www.ncbi.nlm.nih.gov/pubmed/27638249
Genome Biol Evol. 2016 Oct 23;8(10):3140-3148.
Pangenome and Phylogenomic Analysis of the Pathogenic Actinobacterium Rhodococcus equi.
Anastasi E1, MacArthur I1, Scortti M1,2, Alvarez S1, Giguère S3, Vázquez-Boland JA4,2,5.

https://www.ncbi.nlm.nih.gov/pubmed/27924912
Sci Rep. 2016 Dec 7;6:38392. doi: 10.1038/srep38392.
Next-generation systematics: An innovative approach to resolve the structure of complex prokaryotic taxa.
Sangal V1, Goodfellow M2, Jones AL1, Schwalbe EC1, Blom J3, Hoskisson PA4, Sutcliffe IC1.

https://www.ncbi.nlm.nih.gov/pubmed/27074033
PLoS One. 2016 Apr 13;11(4):e0152887. doi: 10.1371/journal.pone.0152887. eCollection 2016.
Plasmid Profiles of Virulent Rhodococcus equi Strains Isolated from Infected Foals in Poland.

https://www.ncbi.nlm.nih.gov/pubmed/25237311
Front Plant Sci. 2014 Aug 19;5:406. doi: 10.3389/fpls.2014.00406. eCollection 2014.
Use of whole genome sequences to develop a molecular phylogenetic framework for Rhodococcus fascians and the Rhodococcus genus.



https://www.ncbi.nlm.nih.gov/pubmed/18179219
J Am Chem Soc. 2008 Jan 30;130(4):1126-7. doi: 10.1021/ja077821p. Epub 2008 Jan 8.
Rhodostreptomycins, antibiotics biosynthesized following horizontal gene transfer from Streptomyces padanus to Rhodococcus fascians.
Kurosawa K1, Ghiviriga I, Sambandan TG, Lessard PA, Barbara JE, Rha C, Sinskey AJ.


	Rhodococcus hoagii 103S	685727	PRJEA41335	41335	Terrabacteria group	Actinobacteria	5.04317	68.8	chromosome:NC_014659.1/FN563149.1	-	1	4649	4540	2010/11/09	2017/05/19	Complete Genome	IREC (International Rhodococcus equi Genome Consortium)	SAMEA2272695	GCA_000196695.1	REFR	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/196/695/GCA_000196695.1_ASM19669v1	20941392	103S

https://www.ncbi.nlm.nih.gov/pubmed/20941392
PLoS Genet. 2010 Sep 30;6(9):e1001145. doi: 10.1371/journal.pgen.1001145.
The genome of a pathogenic rhodococcus: cooptive virulence underpinned by key gene acquisitions.
Letek M1, González P, Macarthur I, Rodríguez H, Freeman TC, Valero-Rello A, Blanco M, Buckley T, Cherevach I, Fahey R, Hapeshi A, Holdstock J, Leadon D, Navas J, Ocampo A, Quail MA, Sanders M, Scortti MM, Prescott JF, Fogarty U, Meijer WG, Parkhill J, Bentley SD, Vázquez-Boland JA.

Figure 3. Schematic overview of relevant metabolic and virulence-related features of R. equi 103S. Complete glycolytic, PPP, and TCA cycle pathways, and all components for aerobic respiration, are present. The TCA cycle incorporates the glyoxylate shunt, which diverts two-carbon metabolites for biosynthesis. The methylcitrate pathway enzymes (pprCBD, REQ09040-60) are also present. The lutABC operon may take over the function of the D-lactate dehydrogenase (cytochrome) REQ00650, which is a pseudogene in 103S. REQ15040 (L-lactate 2-monoxygenase) and REQ27530 (pyruvate dehydrogenase [cytochrome]) can directly convert lactate and pyruvate into acetate. 

### Br-6

http://environmental-microbiology.org/2017/pdfs/要旨集/要旨集_口頭.pdf
既知細菌と比 較して臭素酸還元速度が100倍以上速いRhodococcus sp. Br-6株の分離に成功している。Br-6株は、微好気条件下においての み、臭素酸を臭化物に還元する。このとき、2,6-ジクロロインドフェノール(DCIP)と鉄イオンがそれぞれ酸化還元メディエー ターとして機能していることが明らかになっている。

http://environmental-microbiology.org/2017/pdfs/2014%20Program.pdf
O12-08 Rhodococcus sp. Br-6 株による複数の酸化還元メディエーターを介した臭素酸還元メカニズムの解明https://ci.nii.ac.jp/els/contentscinii_20180102151831.pdf?id=ART0010524150

https://ci.nii.ac.jp/naid/110009974433
CiNii Articles -  PG-003 Rhodococcus equi Br-6株による臭素酸還元メカニズムの解明(生理・代謝・増殖,ポスター発表)
日本微生物生態学会講演要旨集 (29), 125, 2013

https://www.ncbi.nlm.nih.gov/pubmed/27612520
Environ Sci Technol. 2016 Oct 4;50(19):10527-10534. Epub 2016 Sep 23.
Bromate Reduction by Rhodococcus sp. Br-6 in the Presence of Multiple Redox Mediators.
Tamai N1, Ishii T1, Sato Y1, Fujiya H1, Muramatsu Y2, Okabe N2, Amachi S1.

- Analysis of an approximately 1480-bp 16S rRNA gene sequence revealed that the strain is most closely related to Rhodococcus equi, with a sequence similarity of more than 99% (Figure S2). http://pubs.acs.org/doi/suppl/10.1021/acs.est.6b02261/suppl_file/es6b02261_si_001.pdf
- Activity Staining of Diaphorase and Bromate Reductase.
- Figure 6. Activity staining of diaphorase, ferric iron reductase, and bromate reductase
- , suggesting that strain Br-6 has multiple isoforms of diaphorase.
-  (Figure 6). These results strongly suggest that diaphorase is involved in bromate reduction by strain Br-6.

### DLD

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5465047/
3 Biotech. 2017 Jun; 7(2): 153.
Recombinant expression, characterization and application of a dihydrolipoamide dehydrogenase with diaphorase activity from Bacillus sphaericus

- Dihydrolipohyl dehydrogenase (DLD; EC: 1.8.1.4), also known as lipoamide dehydrogenase, is a FAD-dependent enzyme that catalyses the reversible oxidation of dihydrolipoamide to lipoamide (Vaubel et al. 2011). 
- Some of DLD enzymes are also termed diaphorase for their NAD(P)H dehydrogenase activities (Madiagan and Mayhew 1993). 
- we report a strain of B. sphaericus which produced a DLD with NADH-dependent diaphorase activity

誤植
Dihydrolipohyl
->
dihydrolipoyl

### Bromate Reduction

http://pubs.acs.org/doi/abs/10.1021/acs.est.7b00915?journalCode=esthag
Accelerated Reduction of Bromate in Frozen Solution - Environmental Science & Technology (ACS Publications)
Environ. Sci. Technol., 2017, 51 (15), pp 8368–8375

http://pubs.acs.org/doi/full/10.1021/acs.estlett.7b00488
Biological Bromate Reduction Driven by Methane in a Membrane Biofilm Reactor - Environmental Science & Technology Letters (ACS Publications)
Environ. Sci. Technol. Lett., 2017, 4 (12), pp 562–566

----------
## Sinorhizobium
Sinorhizobium (renamed Ensifer)

- https://ja.wikipedia.org/wiki/シノリゾビウム属
- https://en.wikipedia.org/wiki/Sinorhizobium
- https://microbewiki.kenyon.edu/index.php/Sinorhizobium_meliloti
Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Rhizobiazeae; Sinorhizobium (2).

https://twitter.com/search?q=Sinorhizobium

https://www.uniprot.org/taxonomy/382
Rhizobium meliloti (Ensifer meliloti) (Sinorhizobium meliloti)

https://en.wikipedia.org/wiki/Sinorhizobium_meliloti
Synonyms
Rhizobium meliloti Dangeard 1926
Ensifer meliloti (Dangeard 1926) Young 2003

https://www.ncbi.nlm.nih.gov/genome/browse/#!/prokaryotes/Sinorhizobium%20meliloti
Sinorhizobium melilotiには、Repliconが複数ある株（例えば、chromosomeが3つある株、plasmidが5つある株）がいます。

### phylogeny
系統解析

https://www.ncbi.nlm.nih.gov/pubmed/28283520
Appl Environ Microbiol. 2017 May 1;83(10). pii: e03446-16. doi: 10.1128/AEM.03446-16. Print 2017 May 15.
Biogeography of a Novel Ensifer meliloti Clade Associated with the Australian Legume Trigonella suavissima.
Eardly B1, Elia P2, Brockwell J3, Golemboski D4, van Berkum P2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411501/
In our comparative studies, we included representatives of three other Ensifer species (E. medicae, E. arboris, and E. numidicus) that were previously shown to be closely related to E. meliloti (11,–13).

FIG 1
Chromosomal maximum likelihood tree based on concatenated nucleotide sequences of 10 loci in 84 strains.

https://www.ncbi.nlm.nih.gov/pubmed/26380038
Stand Genomic Sci. 2015 Jul 2;10:34. doi: 10.1186/s40793-015-0005-1. eCollection 2015.
High-quality permanent draft genome sequence of Ensifer meliloti strain 4H41, an effective salt- and drought-tolerant microsymbiont of Phaseolus vulgaris.
Mhamdi R1, Ardley J2, Tian R2, Seshadri R3, Reddy TB3, Pati A3, Woyke T3, Markowitz V4, Ivanova N3, Kyrpides N5, Reeve W2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4571125/
In this context, Ensifer meliloti (formerly ‘Sinorhizobium meliloti ’) strain 4H41 was isolated from root nodules of common bean grown in sandy, slightly alkaline soil from the oasis of Rjim-Maatoug in South Tunisia [9].

https://www.ncbi.nlm.nih.gov/pubmed/24976888
Stand Genomic Sci. 2013 Dec 15;9(2):315-24. doi: 10.4056/sigs.4608286. eCollection 2013 Dec 20.
Genome sequence of Ensifer meliloti strain WSM1022; a highly effective microsymbiont of the model legume Medicago truncatula A17.
Terpolilli J1, Hill Y1, Tian R1, Howieson J1, Bräu L2, Goodwin L3, Han J4, Liolios K4, Huntemann M4, Pati A5, Woyke T4, Mavromatis K5, Markowitz V5, Ivanova N3, Kyrpides N3, Reeve W1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4062635/
Two bacterial species, Ensifer meliloti and E. medicae are known to nodulate and fix N2 with Medicago spp. [3], 

https://www.ncbi.nlm.nih.gov/pubmed/22876202
PLoS Genet. 2012;8(8):e1002868. doi: 10.1371/journal.pgen.1002868. Epub 2012 Aug 2.
Population genomics of the facultatively mutualistic bacteria Sinorhizobium meliloti and S. medicae.
Epstein B1, Branca A, Mudge J, Bharti AK, Briskine R, Farmer AD, Sugawara M, Young ND, Sadowsky MJ, Tiffin P.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3410850/
Figure 1
Neighbor-joining trees showing relationships among 32 S. meliloti (blue squares) and 12 S. medicae (red circles).

https://www.ncbi.nlm.nih.gov/pubmed/19656922
Int J Syst Evol Microbiol. 2010 Mar;60(Pt 3):664-74. doi: 10.1099/ijs.0.012088-0. Epub 2009 Aug 5.
Multilocus sequence analysis of root nodule isolates from Lotus arabicus (Senegal), Lotus creticus, Argyrolobium uniflorum and Medicago sativa (Tunisia) and description of Ensifer numidicus sp. nov. and Ensifer garamanticus sp. nov.
Merabet C1, Martens M, Mahdhi M, Zakhia F, Sy A, Le Roux C, Domergue O, Coopman R, Bekki A, Mars M, Willems A, de Lajudie P.
http://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.012088-0#tab2

https://www.ncbi.nlm.nih.gov/pubmed/17329774
Int J Syst Evol Microbiol. 2007 Mar;57(Pt 3):489-503.
Multilocus sequence analysis of Ensifer and related taxa.
Martens M1, Delaere M, Coopman R, De Vos P, Gillis M, Willems A.
http://ijs.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.64344-0#tab2
In this study we used Sinorhizobium (now renamed Ensifer, see below) as a model system to assess the use of MLSA for taxonomic purposes. This genus contains eleven species, most of which have been described in polyphasic studies: Sinorhizobium fredii, Sinorhizobium xinjiangense ( Chen et al., 1988 ), Sinorhizobium meliloti, Sinorhizobium saheli, Sinorhizobium terangae ( de Lajudie et al., 1994 ), Sinorhizobium medicae ( Rome et al., 1996 ), Sinorhizobium arboris, Sinorhizobium kostiense ( Nick et al., 1999 ), Sinorhizobium kummerowiae ( Wei et al., 2002 ), ‘ Sinorhizobium morelense’ ( Wang et al., 2002 ) and Sinorhizobium americanum ( Toledo et al., 2003 ). From 16S rRNA gene comparisons, it is apparent that the monotypic genus Ensifer is highly related to Sinorhizobium ( Balkwill, 2005 ). This organism comprises predatory soil bacteria that were originally described in a separate genus as Ensifer adhaerens, based mainly on morphological and phenotypic characteristics ( Casida, 1982 ). Phylogenetic analyses of 16S rRNA and recA genes ( Balkwill, 2005 ; Willems et al., 2003 ) indicate that Ensifer forms a well-supported cluster together with Sinorhizobium. 

Fig. 2.
Phylogenetic trees based on six genes calculated using the ML method. 
![http://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijs.0.64344-0.F2](http://ijs.microbiologyresearch.org/docserver/fulltext/ijsem/57/3/489fig2.gif)

Fig. 3.
Phylogenetic trees based on concatenated sequences of dnaK, glnA and thrC, calculated using NJ, MP and ML. 
![http://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijs.0.64344-0.F3](http://ijs.microbiologyresearch.org/docserver/fulltext/ijsem/57/3/489fig3.gif)

Fig. 4.
Phylogenetic trees based on concatenated sequences of dnaK, glnA and thrC, gltA, recA and 16S rRNA genes calculated using NJ, MP and ML.
![http://ijs.microbiologyresearch.org/content/ijsem/10.1099/ijs.0.64344-0.F4](http://ijs.microbiologyresearch.org/docserver/fulltext/ijsem/57/3/489fig4.gif)

https://www.ncbi.nlm.nih.gov/pubmed/19120461
FEMS Microbiol Ecol. 2009 Jan;67(1):103-17. doi: 10.1111/j.1574-6941.2008.00590.x.
Rhizobia with different symbiotic efficiencies nodulate Acaciella angustissima in Mexico, including Sinorhizobium chiapanecum sp. nov. which has common symbiotic genes with Sinorhizobium mexicanum.
Rincón-Rosales R1, Lloret L, Ponce E, Martínez-Romero E.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2784085/

- ウイルス、ファージ
https://www.ncbi.nlm.nih.gov/pubmed/28842338
J Struct Biol. 2017 Aug 24. pii: S1047-8477(17)30136-3. doi: 10.1016/j.jsb.2017.08.005. [Epub ahead of print]
Structure, proteome and genome of Sinorhizobium meliloti phage ΦM5: A virus with LUZ24-like morphology and a highly mosaic genome.
Johnson MC1, Sena-Velez M2, Washburn BK3, Platt GN2, Lu S2, Brewer TE2, Lynn JS2, Stroupe ME4, Jones KM5.

- KEYWORDS: antimicrobial peptides, host-microbe interactions, symbiosis
https://www.ncbi.nlm.nih.gov/pubmed/28765224
MBio. 2017 Aug 1;8(4). pii: e01060-17. doi: 10.1128/mBio.01060-17.
Genome-Wide Sensitivity Analysis of the Microsymbiont Sinorhizobium meliloti to Symbiotically Important, Defensin-Like Host Peptides.
Arnold MFF1, Shabab M1, Penterman J1, Boehme KL2, Griffitts JS2, Walker GC3.

- プラスミド、亜ヒ酸酸化
https://www.ncbi.nlm.nih.gov/pubmed/27797963
FEMS Microbiol Ecol. 2017 Jan 1;93(1). doi: 10.1093/femsec/fiw215.
Molecular characterization of the pSinB plasmid of the arsenite oxidizing, metallotolerant Sinorhizobium sp. M14 - insight into the heavy metal resistome of sinorhizobial extrachromosomal replicons.
Romaniuk K1, Dziewit L1, Decewicz P1, Mielnicki S2, Radlinska M3, Drewniak L2.

- 亜ヒ酸酸化細菌 Sinorhizobium sp. Strain A2
https://www.ncbi.nlm.nih.gov/pubmed/26477790
Microbes Environ. 2015;30(4):330-4. doi: 10.1264/jsme2.ME15096. Epub 2015 Oct 17.
Cellular Response of Sinorhizobium sp. Strain A2 during Arsenite Oxidation.
Fukushima K1, Huang H, Hamamura N.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4676556/
The gene encoding TPT in Sinorhizobium spp. (e.g. S. meliloti and S. medicae) is located within the ars operon adjacent to arsC, which encodes arsenate reductase (Fig. 3). 

- プラスミド
https://www.ncbi.nlm.nih.gov/pubmed/26104371
Microbiol Spectr. 2014 Oct;2(5). doi: 10.1128/microbiolspec.PLAS-0005-2013.
The Plasmid Mobilome of the Model Plant-Symbiont Sinorhizobium meliloti: Coming up with New Questions and Answers.
Lagares A, Sanjuán J, Pistorio M.

While plasmids carrying symbiotic functions (pSyms) are known to have high structural stability (approaching that of chromosomes), the remaining plasmid mobilome (referred to as the non-pSym, functionally cryptic, or accessory compartment) has been shown to possess remarkable diversity and to be highly active in conjugation.

- arsenite oxidase (Aio)
https://www.ncbi.nlm.nih.gov/pubmed/22982475
Biochim Biophys Acta. 2013 Feb;1827(2):176-88. doi: 10.1016/j.bbabio.2012.08.007. Epub 2012 Sep 7.
Arsenics as bioenergetic substrates.
van Lis R1, Nitschke W, Duval S, Schoepp-Cothenet B.
Fig. 2. Organization of observed prokaryotic aio gene clusters. Organizations a–g are from demonstrated AsIII-oxidizing Bacteria. a: Sinorhizobium sp. str. M14, 

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/28386109
Sci Rep. 2017 Apr 6;7(1):675. doi: 10.1038/s41598-017-00730-7.
The underlying process of early ecological and genetic differentiation in a facultative mutualistic Sinorhizobium meliloti population.
Toro N1, Villadas PJ2, Molina-Sánchez MD2, Navarro-Gómez P3, Vinardell JM3, Cuesta-Berrio L4, Rodríguez-Carvajal MA4.

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/28138345
J Genomics. 2017 Jan 18;5:12-15. doi: 10.7150/jgen.17863. eCollection 2017.
Genomic characterization of eight Ensifer strains isolated from pristine caves and a whole genome phylogeny of Ensifer (Sinorhizobium).
Kumar HK1, Gan HM1, Tan MH1, Eng WW1, Barton HA2, Hudson AO3, Savka MA3.

- 比較ゲノム
https://www.ncbi.nlm.nih.gov/pubmed/23425606
Genome Biol. 2013 Feb 20;14(2):R17. doi: 10.1186/gb-2013-14-2-r17.
Comparative genomics of the core and accessory genomes of 48 Sinorhizobium strains comprising five genospecies.
Sugawara M, Epstein B, Badgley BD, Unno T, Xu L, Reese J, Gyaneshwar P, Denny R, Mudge J, Bharti AK, Farmer AD, May GD, Woodward JE, Médigue C, Vallenet D, Lajus A, Rouy Z, Martinez-Vaz B, Tiffin P, Young ND, Sadowsky MJ.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/
Figure 1
Neighbor-joining tree based on concatenated sequences for 645 protein coding genes.

the outgroup (Rhizobium leguminosarum bv. trifolii WSM1325).
The tree was rooted by treating the R. leguminosarum strain as an outgroup

![https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/figure/F1/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/bin/gb-2013-14-2-r17-1.jpg)

- プラスミド
Nucleic Acids Res. 2008 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/ "For example, Sinorhizobium meliloti megaplasmid pSymB (1 683 333 bp) is not self-transmissible, and has not been successfully cured from S. meliloti (51). It has been suggested that pSymB has acquired genes essential to the host's viability and that its genomic signature became similar to that of the main chromosome due to long-term residence in that host (20)."

https://www.ncbi.nlm.nih.gov/pubmed/16199569
J Bacteriol. 2005 Oct;187(20):6991-7.
Novel pathway for arsenic detoxification in the legume symbiont Sinorhizobium meliloti.
Yang HC1, Cheng J, Finan TM, Rosen BP, Bhattacharjee H.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251620/
The two biologically relevant oxidation states of inorganic arsenic are arsenite [As(III)] and arsenate [As(V)], the former being more toxic than the later. 

- コドン使用
https://www.ncbi.nlm.nih.gov/pubmed/14597394
Gene. 2003 Nov 27;320:109-16.
The strength of translational selection for codon usage varies in the three replicons of Sinorhizobium meliloti.
Peixoto L1, Zavala A, Romero H, Musto H.

Sinorhizobium melilotiの染色体とプラスミドpSymBのコドン使用は、翻訳過程を効率化する自然選択（translational selection）の寄与が大きいのに対して、（根粒形成と窒素固定に関与する）プラスミドpSymAのコドン使用は、変異バイアス（mutational bias）の寄与が大きいと考えられる。

- 二連続塩基組成
https://www.ncbi.nlm.nih.gov/pubmed/12444420
Funct Integr Genomics. 2002 Nov;2(6):274-81. Epub 2002 Aug 1.
Dinucleotide compositional analysis of Sinorhizobium meliloti using the genome signature: distinguishing chromosomes and plasmids.
Wong K1, Finan TM, Golding GB.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/
プラスミドpSymBの二連続塩基組成（genomic signature）は、染色体のそれと類似しているので、宿主に長期間滞在したと考えられる。



考察

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053727/figure/F1/
この研究では、Rhizobium leguminosarum を外群(outgroup)として用いて系統樹に根をつけている。
この645遺伝子配列の結合データ（concatenated sequences）を用いた系統樹では、
先ず、S. terangaeと他のグループが分岐し、
次に、S. frediiと他のグループが分岐し、
さらに、S. saheliと他のグループが分岐し、
最後に、Sinorhizobium medicaeクレードとSinorhizobium melilotiクレードとが分岐している。

一方、 https://www.ncbi.nlm.nih.gov/pubmed/17329774 で複数の系統樹（Fig. 2-4. Phylogenetic trees）の樹形を確認したところ、Sinorhizobium/Ensifer属の中でE. frediiとE. saheliの何れがより祖先的な生物から分岐して誕生したのか（外群になるのか）判断できない。例えば、
Fig. 2. "thrC ML"
Fig. 4. "6 genes MP"
と他の系統樹で、E. fredii、E. saheli、3株（E. meliloti, E. medicae, E. arboris）の系統関係が一致しない。

例えば、どの内群のグループよりも遠縁である E. adhaerens を外群として選択する。

http://nesseiken.info/Chiba_lab/index.php?授業%2FH24%2F進化生物学I%2F系統樹に関する基本用語
- 内群（ingroup）　今、系統推定の対象としているグループのこと。
- 外群 (outgoup)　内群に含まれない分類群はすべて外群（outgroup）になる。外群は通常、系統樹に根をつけるときに使われ、内群の姉妹群から複数のものを用いることが多い。


https://ja.wikipedia.org/wiki/外群
外群は系統樹に根をつけ、系統関係を知りたい内群の系統的位置関係を決定するのに用いられる。形質変化の極性は有根系統樹のみから決定できる[1]ため、外群の選択は系統に沿った形質状態の進化を理解する上で必須である。
外群は、内群の他のグループに近縁であるが、しかし他のどの内群のグループよりも互いに遠縁であると考えられるものを選択する。これらの系統関係から導かれた進化学的結論は、外群の種は内群と共通の祖先をもち、それは内群の各グループの共通祖先よりも古いものであるということである。外群は内群の姉妹群、またはより遠縁なものである[2]。

----------
### KGO-5

http://www.arsenic-sci-soc.jp/18th%20symposium.pdf
第 18 回ヒ素シンポジウム 講演要旨集開催日: 2012年11月24日(土)・25日(日)国内土壌より分離された亜ヒ酸酸化細菌 KGO-5 株 ◯董 旦、天知誠吾(千葉大院・園芸・応用生命化学)土壌中のヒ素は主に無機態ヒ素である亜ヒ酸[As(III)]とヒ酸[As(V)]として存在する。
得られた菌株をKGO-5株とし、その亜ヒ酸酸化能を液体培地で確認した。
亜ヒ酸を唯一のエネルギー源として含む集積培養系より、KGO-5株を分離した。KGO-5株はグラム陰性の桿菌で、16S rRNA 遺伝子に基づく系統解析より、根粒菌 の1種であるSinorhizobium(シノリゾビウム)に近縁(98%~99%相同)であるこ とがわかった(図1)。また、SDB1株(Lugtu et al., 2009)やSinorhizobium sp. DAO10 株(Rhine et al., 2006)など既知の亜ヒ酸酸化細菌とも近縁であった。
https://www.uniprot.org/uniprot/A3EWK7
aroA - Arsenite oxidase - Sinorhizobium sp. DAO10 - aroA gene & protein
https://www.ncbi.nlm.nih.gov/protein/116090303
arsenite oxidase, partial [Sinorhizobium sp. DAO10]
https://www.ncbi.nlm.nih.gov/nuccore/?term=Sinorhizobium%20sp.%20DAO10
Sinorhizobium sp. DAO10 arsenite oxidase (aroA) gene, partial cds
Sinorhizobium sp. DAO10 16S ribosomal RNA gene, partial sequence


https://core.ac.uk/download/pdf/96948362.pdf
博士論文土壌環境における亜ヒ酸の挙動に及ぼす細菌群集の影響
2-6-1 供試菌株供試菌株として Alphaproteobacteria 綱に属する亜ヒ酸酸化細菌 Shinorhizobium sp. KGO-5 (Dong et al., 2014)を用いた。KGO-5 株は本研究室にお いて高度ヒ素汚染土壌から単離された通性独立栄養性の亜ヒ酸酸化細菌である。2-6-2 KGO-5 株の培養及び菌体懸濁液の調製
----------

## Rickettsiales
- https://ja.wikipedia.org/wiki/リケッチア目
- https://en.wikipedia.org/wiki/Holosporaceae
- https://ja.wikipedia.org/wiki/ホロスポラ属
- https://ja.wikipedia.org/wiki/ケディバクター属

https://ja.wikipedia.org/wiki/ミディクロリア
(Candidatus Midichloria mitochondrii)はミトコンドリアに細胞内共生するリケッチアであり、マダニの卵巣に見出される。この名は映画スター・ウォーズ・シリーズに登場しフォースに寄与する共生微生物ミディ＝クロリアンに因んでおり、監督ジョージ・ルーカスの公認を得て命名されている[1]。

http://www.nikkei-science.com/?p=46394
ミトコンドリアの起源に新説〜日経サイエンス2015年4月号より
リケッチアが祖先である可能性が浮上

https://www.ncbi.nlm.nih.gov/pubmed/25609566
Sci Rep. 2015 Jan 22;5:7949. doi: 10.1038/srep07949.
An integrated phylogenomic approach toward pinpointing the origin of mitochondria.
Wang Z1, Wu M1.

https://www.ncbi.nlm.nih.gov/pubmed/25333787
PLoS One. 2014 Oct 15;9(10):e110685. doi: 10.1371/journal.pone.0110685. eCollection 2014.
Phylogenomic reconstruction indicates mitochondrial ancestor was an energy parasite.
Wang Z1, Wu M1.

https://www.ncbi.nlm.nih.gov/pubmed/16703114
PLoS Genet. 2006 May;2(5):e76. Epub 2006 May 12.
Genome sequence of Rickettsia bellii illuminates the role of amoebae in gene exchanges between intracellular pathogens.
Ogata H1, La Scola B, Audic S, Renesto P, Blanc G, Robert C, Fournier PE, Claverie JM, Raoult D.

----------
## Wolbachia
https://ja.wikipedia.org/wiki/ボルバキア

https://kotobank.jp/word/ボルバキア-669591

Nov 29, 2017
https://www.youtube.com/watch?v=S2tND3OmwzY
Mosquitoes Might SAVE Lives, Thanks To Bacteria | I Contain Multitudes - YouTube

2017年3月9日 
https://www.natureasia.com/ja-jp/nature/highlights/83856
微生物学： ボルバキア菌による昆虫の生存操作

ボルバキア由来のプロファージWOがコードする2つの遺伝子が、細胞質不和合を再現する

2016年10月12日 
https://www.natureasia.com/ja-jp/ncomms/pr-highlights/11040
【進化】細菌に感染するウイルスがクロゴケグモと同じDNAを持っていた
ファージWO（ボルバキア菌に感染するウイルス）のゲノム配列を解読し、そのゲノムの一部を構成する遺伝子が真核生物の遺伝子に類似していることを発見した。

2015年9月
http://www.nikkei-science.com/201509_046.html
デング熱ストッパー | 日経サイエンス　
自然界によく見られる細菌の助けを借りて，蚊によるデング熱の媒介を阻止する試みが始まっている。　

2014 陰山大輔
https://www.jstage.jst.go.jp/article/konchubiotec/83/3/83_3_243/_pdf
昆虫の生殖を操作する細胞内共生細菌 Wolbachia の機能と特徴

http://www.aist.go.jp/aist_j/press_release/pr2014/pr20140701/pr20140701.html
産総研：トコジラミの生存を支える共生細菌ボルバキアのビタミンB7合成能力

http://www.nikkei-science.com/page/magazine/1006/201006_062.html
細菌によって変わる虫たち

2005年9月1日
http://www.natureasia.com/ja-jp/reviews/highlight/10933
不和合性の解析：ボルバキア(Wolbachia)を対象にする

2002年7月
http://www.nikkei-science.com/page/sci_book/bessatu/51221.html
細菌が操る性転換 | 日経サイエンス　
昆虫などに広く寄生するボルバキア属の細菌は，寄生した相手を雄から雌に性転換させたり雄になる卵を殺したりする。また寄生相手の交配を妨害することさえある。

----------
### Amachi Seigo
天知 誠吾

https://www.ncbi.nlm.nih.gov/pubmed/28529651
J Genomics. 2017 Apr 6;5:51-53. doi: 10.7150/jgen.19846. eCollection 2017.
Draft Genome Sequence of Roseovarius sp. A-2, an Iodide-Oxidizing Bacterium Isolated from Natural Gas Brine Water, Chiba, Japan.
Yuliana T, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.

https://www.ncbi.nlm.nih.gov/pubmed/27738047
Genome Announc. 2016 Oct 13;4(5). pii: e01155-16. doi: 10.1128/genomeA.01155-16.
Draft Genome Sequence of Arenibacter sp. Strain C-21, an Iodine-Accumulating Bacterium Isolated from Surface Marine Sediment.
Ito K, Nakajima N, Yamamura S, Tomita M, Suzuki H, Amachi S.

http://www.ncbi.nlm.nih.gov/pubmed/25977440
Genome Announc. 2015 May 14;3(3). pii: e00472-15. doi: 10.1128/genomeA.00472-15.
Draft Genome Sequence of Anaeromyxobacter sp. Strain PSR-1, an Arsenate-Respiring Bacterium Isolated from Arsenic-Contaminated Soil.
Tonomura M, Ehara A, Suzuki H, Amachi S.

http://www.ncbi.nlm.nih.gov/pubmed/25635012
Genome Announc. 2015 Jan 29;3(1). pii: e01478-14. doi: 10.1128/genomeA.01478-14.
Draft Genome Sequence of Geobacter sp. Strain OR-1, an Arsenate-Respiring Bacterium Isolated from Japanese Paddy Soil.
Ehara A, Suzuki H, Amachi S.

http://www.ncbi.nlm.nih.gov/pubmed/24994802
Genome Announc. 2014 Jul 3;2(4). pii: e00659-14. doi: 10.1128/genomeA.00659-14.
Draft genome sequence of strain Q-1, an iodide-oxidizing alphaproteobacterium isolated from natural gas brine water.
Ehara A, Suzuki H, Kanesaki Y, Yoshikawa H, Amachi S.




https://www.ncbi.nlm.nih.gov/pubmed/27612520
Environ Sci Technol. 2016 Oct 4;50(19):10527-10534. Epub 2016 Sep 23.
Bromate Reduction by Rhodococcus sp. Br-6 in the Presence of Multiple Redox Mediators.
Tamai N1, Ishii T1, Sato Y1, Fujiya H1, Muramatsu Y2, Okabe N2, Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/26041311
Biosci Biotechnol Biochem. 2015;79(11):1898-905. doi: 10.1080/09168451.2015.1052767. Epub 2015 Jun 4.
A putative multicopper oxidase, IoxA, is involved in iodide oxidation by Roseovarius sp. strain A-2.
Shiroyama K1, Kawasaki Y1, Unno Y2, Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/25051896
Biosci Biotechnol Biochem. 2014;78(11):1963-70. doi: 10.1080/09168451.2014.940276. Epub 2014 Jul 23.
Arsenite oxidation by a facultative chemolithoautotrophic Sinorhizobium sp. KGO-5 isolated from arsenic-contaminated soil.
Dong D1, Ohtsuka T, Dong DT, Amachi S.

Mini Reviews
https://www.ncbi.nlm.nih.gov/pubmed/21558718
Microbes Environ. 2008;23(4):269-76.
Microbial contribution to global iodine cycling: volatilization, accumulation, reduction, oxidation, and sorption of iodine.
Amachi S1.

https://www.ncbi.nlm.nih.gov/pubmed/17933915
Appl Environ Microbiol. 2007 Dec;73(23):7536-41. Epub 2007 Oct 12.
Hydrogen peroxide-dependent uptake of iodine by marine Flavobacteriaceae bacterium strain C-21.
Amachi S1, Kimura K, Muramatsu Y, Shinoyama H, Fujii T.

https://www.ncbi.nlm.nih.gov/pubmed/17644635
Appl Environ Microbiol. 2007 Sep;73(18):5725-30. Epub 2007 Jul 20.
Dissimilatory iodate reduction by marine Pseudomonas sp. strain SCT.
Amachi S1, Kawaguchi N, Muramatsu Y, Tsuchiya S, Watanabe Y, Shinoyama H, Fujii T.

2016年6月3日
https://www.amano-enzyme.co.jp/jp/company/17_pro.html
新規マルチ銅オキシダーゼIOXを活用した酵素殺菌剤の殺菌メカニズムの解明
https://www.amano-enzyme.co.jp/jp/company/pdf/17/pg05.pdf
Q-1 株

2014年5月26日
https://kaken.nii.ac.jp/file/KAKENHI-PROJECT-23580103/23580103seika.pdf
ヒ酸還元能を持つジオバクター属細菌による水田土壌からのヒ素溶出
OR-1株のゲノムにはヒ素代謝遺伝子群を コードする領域(Arsenic island)が独立して2つ存在し、異化的ヒ酸還元反応を触媒すると考えられるarrAB遺伝子 も見出された。

2012.12.13
http://tenbou.nies.go.jp/news/jnews/detail.php?i=9992
千葉大学、微生物酵素による放射性ヨウ素の土壌中での安定化プロセスを解明



http://www.jsmrs.jp/journal/No32_2/No32_2_115.pdf
Microb. Resour. Syst., 2016
総説
ヨウ素酸化細菌 ─その分離から応用まで─

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/5/51_286/_article/-char/ja/
化学と生物 Vol. 51, No. 5, 2013
微生物による無機ヨウ素化合物の酸化還元反応
https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/5/51_286/_pdf
図5■ Pseudomonas sp. SCT株のヨウ素酸呼吸時に働く推定呼吸鎖コンポーネント

https://ci.nii.ac.jp/naid/110009767287/
地球化学 Chikyukagaku 47(4), 209-219, 2013
ヨウ素の地球化学と微生物 : ヨウ素の揮発,濃縮,酸化,還元,吸着,脱ハロゲン化反応を触媒するバクテリア
http://www.geochem.jp/journal_j/contents/pdf/47-4-209.pdf
- これらの結果から,海水中のヨウ素酸は海洋生物 の有する硝酸還元酵素によってヨウ化物イオンに還元 されるという考え方が広く受け入れられるようになっ た。しかしながら,
- 筆者らは,硝酸還元酵素ではなく,ヨウ素酸に特異 的な還元酵素を有するバクテリアも存在するのではな いかと考え,海洋環境からの分離を試みた。スクリー ニングの結果,相模湾の堆積物から Pseudomonas sp. SCT 株を分離した(Amachi et al., 2007a)。

天知誠吾 著 - ‎2010
https://www.jstage.jst.go.jp/article/geochemproc/57/0/57_0_61/_pdf
ヨウ素の水圏循環に与える微生物の影響
SCT 株


----------
