Haruo Suzuki (haruo[at]g-language[dot]org)  
Last Update: 2018-05-17


----------

# microbiome
**微生物群集**

## Table of Contents
- [featuring](#featuring)
- [16S](#16s)
- [amplicon-shotgun](#amplicon-shotgun)
- [mothur-qiime](#mothur-qiime)
- [American Gut](#american-gut)
- [Artificial sweeteners](#artificial-sweeteners)
- [Earth Microbiome Project](#emp)
- [GWAS](#gwas)
- [meta-analysis](#meta-analysis)

----------

https://github.com/haruosuz/metasub
MetaSUB: Metagenomics & Metadesign of Subways & Urban Biomes
都市の微生物群集

----------
## Updates

----------



https://www.ncbi.nlm.nih.gov/pubmed/29460198
J Plant Res. 2018 Jul;131(4):709-717. doi: 10.1007/s10265-018-1017-x. Epub 2018 Feb 19.
Collaborative environmental DNA sampling from petal surfaces of flowering cherry Cerasus × yedoensis 'Somei-yoshino' across the Japanese archipelago.
Ohta T1, Kawashima T2, Shinozaki NO3, Dobashi A4, Hiraoka S5, Hoshino T6, Kanno K7, Kataoka T8, Kawashima S9, Matsui M10, Nemoto W11, Nishijima S12,13,14, Suganuma N15, Suzuki H16, Taguchi YH17, Takenaka Y18, Tanigawa Y19, Tsuneyoshi M20, Yoshitake K21, Sato Y22, Yamashita R22, Arakawa K23, Iwasaki W10.


----------
## featuring


----------
## Knight R

https://knightlab.ucsd.edu/wordpress/?page_id=25

https://www.ncbi.nlm.nih.gov/pubmed/29795328
Nat Rev Microbiol. 2018 Jul;16(7):410-422. doi: 10.1038/s41579-018-0029-9.
Best practices for analysing microbiomes.
Knight R1,2,3, Vrbanac A4, Taylor BC4, Aksenov A5, Callewaert C6,7, Debelius J6, Gonzalez A6, Kosciolek T6, McCall LI5, McDonald D6, Melnik AV5, Morton JT6,8, Navas J8, Quinn RA5, Sanders JG6, Swafford AD9, Thompson LR10,11, Tripathi A12, Xu ZZ6, Zaneveld JR13, Zhu Q6, Caporaso JG14, Dorrestein PC9,5,6.

https://www.ncbi.nlm.nih.gov/pubmed/29795540
Nat Microbiol. 2018 Jun;3(6):652-661. doi: 10.1038/s41564-018-0156-0. Epub 2018 May 24.
Methods for phylogenetic analysis of microbiome data.
Washburne AD1, Morton JT2,3, Sanders J3, McDonald D3, Zhu Q3, Oliverio AM4,5, Knight R2,3.

----------
## Mason CE

https://www.ncbi.nlm.nih.gov/pubmed/28337071
J Biomol Tech. 2017 Apr;28(1):8-18. doi: 10.7171/jbt.17-2801-006. Epub 2017 Mar 17.
International Standards for Genomes, Transcriptomes, and Metagenomes.
Mason CE1, Afshinnekoo E2, Tighe S3, Wu S4, Levy S5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5359768/table/T3/
TABLE 3
Comparison of metagenomic assay capabilities and limitations

https://www.nature.com/articles/s41598-017-06665-3
Large-scale differences in microbial biodiversity discovery between 16S amplicon and shotgun sequencing | Scientific Reports

https://www.ncbi.nlm.nih.gov/pubmed/29930288
Nat Rev Microbiol. 2018 Jun 21. doi: 10.1038/s41579-018-0047-7. [Epub ahead of print]
Surveying what's flushed away.
Iraola G1,2,3, Kumar N3.
https://www.nature.com/articles/s41579-018-0047-7

https://www.ncbi.nlm.nih.gov/pubmed/29890228
Drug Discov Today. 2018 Jun 8. pii: S1359-6446(17)30357-4. doi: 10.1016/j.drudis.2018.06.005. [Epub ahead of print]
Interrogating the microbiome: experimental and computational considerations in support of study reproducibility.
Poussin C1, Sierro N1, Boué S1, Battey J1, Scotti E1, Belcastro V1, Peitsch MC1, Ivanov NV1, Hoeng J2.
Review
Informatics

https://link.springer.com/protocol/10.1007/978-1-4939-7463-4_8
Comparative Metagenomics | SpringerLink
Andrew Maltez ThomasFelipe Prata LimaLivia Maria Silva MouraAline Maria da SilvaEmmanuel Dias-NetoJoão C. SetubalEmail author
Protocol
First Online: 26 December 2017

https://www.ncbi.nlm.nih.gov/pubmed/28978331
Microbiome. 2017 Oct 4;5(1):132. doi: 10.1186/s40168-017-0349-4.
Communicating the promise, risks, and ethics of large-scale, open space microbiome and metagenome research.
Shamarina D1, Stoyantcheva I1, Mason CE2,3,4, Bibby K5, Elhaik E6.

https://www.ncbi.nlm.nih.gov/pubmed/28934964
Genome Biol. 2017 Sep 21;18(1):182. doi: 10.1186/s13059-017-1299-7.
Comprehensive benchmarking and ensemble approaches for metagenomic classifiers.
McIntyre ABR1,2,3, Ounit R4, Afshinnekoo E2,3,5, Prill RJ6, Hénaff E2,3, Alexander N2,3, Minot SS7, Danko D1,2,3, Foox J2,3, Ahsanuddin S2,3, Tighe S8, Hasan NA9,10, Subramanian P9, Moffat K9, Levy S11, Lonardi S4, Greenfield N7, Colwell RR9,12, Rosen GL13, Mason CE14,15,16.


https://www.ncbi.nlm.nih.gov/pubmed/28349976
Nat Microbiol. 2017 Mar 28;2:16275. doi: 10.1038/nmicrobiol.2016.275.
Geomicrobiology of the built environment.
Gadd GM1.
http://www.nature.com/articles/nmicrobiol2016275

https://www.ncbi.nlm.nih.gov/pubmed/27941038
Br Med Bull. 2016 Dec;120(1):27-33. Epub 2016 Nov 23.
Globalizing and crowdsourcing biomedical research.
Afshinnekoo E1,2,3, Ahsanuddin S1,2,4, Mason CE5,2,6.

----------
## amplicon-shotgun

targeted amplicon sequencing (16S, 18S, ITS) vs. shotgun sequencing (metagenomics)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5359768/table/T3/
TABLE 3
Comparison of metagenomic assay capabilities and limitations

31 July 2017
https://www.nature.com/articles/s41598-017-06665-3
Large-scale differences in microbial biodiversity discovery between 16S amplicon and shotgun sequencing

https://www.ncbi.nlm.nih.gov/pubmed/21121044
Pac Symp Biocomput. 2011:165-76.
Comparing bacterial communities inferred from 16S rRNA gene sequencing and shotgun metagenomics.
Shah N1, Tang H, Doak TG, Ye Y.

----------
## mothur-qiime
https://mothur.org

https://www.pediatricsurgery.site/entry/2017/11/19/200241
mothur開発者によるmothurとQIIMEの比較 - Note of Pediatric Surgery

https://biosciencedbc.jp/gadget/human/170831_mori_170830.pdf
メタ16S解析利点
・マシンパワーは少なくて済み、解析ツールも普及(QIIME・mothur等)

https://www.jstage.jst.go.jp/article/jim/29/3/29_135/_article/-char/ja/
総　　説 ＜特集：腸内細菌叢Microbiotaの分子生物学的解析法の成果と未来＞
進化する次世代シーケンサーによる腸内細菌叢の解析
東 佳那子, 中山 二郎
2015 年 29 巻 3 号 p. 135-144
https://www.jstage.jst.go.jp/article/jim/29/3/29_135/_pdf
データセットは,16S rRNA 配列から菌叢データを 得るためのオープンソースソフトウェアを開発してい る Mothur プロジェクト(4)が,Silva の 16S rRNA の全長配列データベースから 14,956 本を網羅的に抽出 し作成したデータセット silva.bacteria.fasta を用いた(http://www.mothur.org/wiki/Silva_reference_files).

ここで は,Mothur 同様,菌叢解析のオープンソースソフトウェ アである QIIME(Quantitative Insights into Microbial Ecology, http://qiime.org/)(5) の assign_taxonomy.py のプログラムの uclust のアルゴリズムを用いた.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3775626/
PeerJ. 2013; 1: e157.
Abundance-weighted phylogenetic diversity measures distinguish microbial community states and are robust to sampling depth

In microbial ecology studies, the most commonly used ways of investigating alpha (within-sample) diversity are either to apply non-phylogenetic measures such as Simpson’s index to Operational Taxonomic Unit (OTU) groupings, or to use classical phylogenetic diversity (PD), which is not abundance-weighted. 

As of the publication of this paper, no abundance-weighted phylogenetic alpha diversity measures are implemented in either mothur (Schloss et al., 2009) or QIIME (Caporaso et al., 2010), two of the most popular tools for analysis of microbial ecology data. 


----------
## 16S

https://www.ncbi.nlm.nih.gov/pubmed/29688343
FEMS Microbiol Lett. 2018 May 1;365(10). doi: 10.1093/femsle/fny104.
A town on fire! Integrating 16S rRNA gene amplicon analyses into an undergraduate microbiology lecture class.
Tobin TC1, Shade A2.
In this module, microbiology students used Quantitative Insights into Microbial Ecology (QIIME) to perform taxonomic, phylogenetic and statistical analyses on bacterial communities from three hot mine fire-impacted surface soils using 16S rRNA gene amplicon sequences. 

https://twitter.com/BioMickWatson/status/968147242401058817
Mick Watson on Twitter: "If you're working in 16S microbiome research, please read our paper which tries to suggest best practices! https://t.co/XWVRxUKFRW"
The madness of microbiome: Attempting to find consensus “best practice” for 16S microbiome studies
10:34 AM - 26 Feb 2018

https://www.ncbi.nlm.nih.gov/pubmed/29427429
Appl Environ Microbiol. 2018 Mar 19;84(7). pii: e02627-17. doi: 10.1128/AEM.02627-17. Print 2018 Apr 1.
The Madness of Microbiome: Attempting To Find Consensus "Best Practice" for 16S Microbiome Studies.
Pollock J#1,2, Glendinning L#2, Wisedchanwet T2, Watson M2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861821/
Of the three most commonly used alignments which are guided by secondary structure (i.e., Greengenes [73], RDP [74], and SILVA [75]), the Greengenes alignment was observed to be of poor quality, leading to significantly greater richness and diversity estimates.

https://www.ncbi.nlm.nih.gov/pubmed/28361695
BMC Genomics. 2017 Mar 14;18(Suppl 2):114. doi: 10.1186/s12864-017-3501-4.
SILVA, RDP, Greengenes, NCBI and OTT - how do these taxonomies compare?
Balvočiūtė M1, Huson DH2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5374703/
For the comparisons we used a taxonomy associated with the Greengenes database as released on May 2013. Although Greengenes is still included in some metagenomic analyses packages, for example QIIME [22], it has not been updated for the last three years.

https://www.pediatricsurgery.site/entry/2017/11/19/200241
mothur開発者によるmothurとQIIMEの比較 - Note of Pediatric Surgery

mothurではRDP、SILVA、greengenesを使用することができる

Mar 2017
https://forum.qiime2.org/t/classification-using-greengenes-v-silva/409
Classification using GreenGenes v. SILVA - User support - QIIME 2 Forum

http://blog.mothur.org/2015/08/04/No-greengenes-hasnt-improved/
No, greengenes' reference alignment hasn't improved

- SILVA (good for alignment and classification)
- greengenes (good for alignment and classification)
- RDP (good for classification)

https://www.researchgate.net/post/Is_Greengenes_or_SILVA_better_for_bacterial_microbiome_studies
Is Greengenes or SILVA better for bacterial microbiome studies?

https://www.ncbi.nlm.nih.gov/pubmed/27166378
Nucleic Acids Res. 2016 Jun 20;44(11):5022-33. doi: 10.1093/nar/gkw396. Epub 2016 May 10.
Phylogeny-aware identification and correction of taxonomically mislabeled sequences.
Kozlov AM1, Zhang J2, Yilmaz P3, Glöckner FO4, Stamatakis A5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914121/
First, we evaluate taxonomic annotations of type strains only, using the same sequence set and alignment (LTP v123) for all four databases (data sets GG13_T, LTP123_T, RDP11_T and SLV123_T in Table ​Table1).1). 

https://www.ncbi.nlm.nih.gov/pubmed/23460914
PLoS One. 2013;8(2):e57923. doi: 10.1371/journal.pone.0057923. Epub 2013 Feb 27.
The variability of the 16S rRNA gene in bacterial genomes and its consequences for bacterial community analyses.
Větrovský T1, Baldrian P.


Alexey M. Kozlov, 1 ,* Jiajie Zhang, 1 Pelin Yilmaz, 2 Frank Oliver Glöckner, 2 , 3 and Alexandros Stamatakis 1 , 4
1The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 69118 Heidelberg, Germany
2Microbial Genomics and Bioinformatics Research Group, Max Planck Institute for Marine Microbiology, 28359 Bremen, Germany
3Jacobs University Bremen gGmbH, Campus Ring 1, 28759 Bremen, Germany
4Karlsruhe Institute of Technology, Institute for Theoretical Informatics, Postfach 6980, 76128 Karlsruhe, Germany
*To whom correspondence should be addressed. Tel: +49 6221533291; Fax: +49 6221533298; Email: gro.sti-h@volzoK.yexelA

----------
## Greengenes

http://greengenes.secondgenome.com
The Greengenes Database

http://greengenes.secondgenome.com/downloads
Download Area

----------
## RDP

https://www.ncbi.nlm.nih.gov/pubmed/24288368
Nucleic Acids Res. 2014 Jan;42(Database issue):D633-42. doi: 10.1093/nar/gkt1244. Epub 2013 Nov 27.
Ribosomal Database Project: data and tools for high throughput rRNA analysis.
Cole JR1, Wang Q, Fish JA, Chai B, McGarrell DM, Sun Y, Brown CT, Porras-Alfaro A, Kuske CR, Tiedje JM.

http://rdp.cme.msu.edu
RDP Release 11 -- Sequence Analysis Tools

http://rdp.cme.msu.edu/misc/resources.jsp
RDP Resource Download Area

----------

https://ja.wikipedia.org/wiki/16S_rRNA系統解析

http://www.aist.go.jp/aist_j/press_release/pr2012/pr20121030/pr20121030.html
産総研：生物種を越えた16S rRNA遺伝子の機能相補性を確認

https://www.jstage.jst.go.jp/article/jslab/23/1/23_24/_pdf
16S rRNA 遺伝子の大量シークェンシングによる 菌叢解析の現状と問題点
是則有希 著 - ‎2012

http://www.jarmam.gr.jp/situmon3/16sr-rna.html
16S rRNAでの微生物同定






----------
## American Gut 

May 15, 2018
https://www.sciencedaily.com/releases/2018/05/180515092931.htm
Big data from world's largest citizen science microbiome project serves food for thought
How factors such as diet, antibiotics and mental health status can influence the microbial and molecular makeup of your gut

http://msystems.asm.org/content/3/3/e00031-18
American Gut: an Open Platform for Citizen Science Microbiome Research

2013.01.07
http://natgeo.nikkeibp.co.jp/nng/article/news/14/7342/
人の消化管に棲む微生物を大規模調査 | ナショナルジオグラフィック日本版サイト

----------
## Artificial sweeteners

https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/publicaton_metagenome_for-human-health-j.pdf
[PDF]ヒトの健康における 細菌およびメタゲノム - イルミナ株式会社
例えば、特定の人工甘味料はマウスの腸管を乱れさせ、耐 糖能を減少させます 28。

2017年6月9日
https://www.alic.go.jp/joho-s/joho07_001494.html
人工甘味料と糖代謝｜農畜産業振興機構

2016-09-06
https://unlog.me/topics/05theses-artificial-sweeteners
人工甘味料は腸内細菌を変化させ、血糖値が下がりにくくなる

2014年9月19日
http://aasj.jp/news/watch/2190
人工甘味料による糖尿（Nature オンライン版掲載論文）

https://www.natureasia.com/ja-jp/nature/514/7521/
微生物学：人工甘味料は腸内微生物相を変化させることで耐糖能異常を引き起こす
https://www.ncbi.nlm.nih.gov/pubmed/25231862
Nature. 2014 Oct 9;514(7521):181-6. doi: 10.1038/nature13793. Epub 2014 Sep 17.
Artificial sweeteners induce glucose intolerance by altering the gut microbiota.


----------
## EMP

2017.12.14
https://wired.jp/2017/12/14/catalogue-every-microbe/
地球上の無数の細菌をすべてカタログ化──壮大なプロジェクトが目指すもの
地球上の微生物の一覧をつくるプロジェクト「アース・マイクロバイオーム・プロジェクト（EMP）」。

2017年11月2日
https://www.natureasia.com/ja-jp/nature/pr-highlights/12252
【微生物学】地球における微生物多様性に関するデータのクラウドソーシング
このメタ分析は、地球マイクロバイオームプロジェクト（EMP）の第一段階の一環として実施された。EMPは、地球上の微生物の特徴を明らかにすることを目標としている。

5 Nov 2017
https://twitter.com/NatureDigest/status/927029560788500481
Nature ダイジェスト、編集部 on Twitter: "地球上の微生物の多様性評価のために2007年に発足したEMP（Earth Microbiome Project）から最初の報告。論文はオープンです。 doi:10.1038/nature24621 https://t.co/2BpJiFIr7F"

https://www.ncbi.nlm.nih.gov/pubmed/29088705
Nature. 2017 Nov 23;551(7681):457-463. doi: 10.1038/nature24621. Epub 2017 Nov 1.
A communal catalogue reveals Earth's multiscale microbial diversity.
Earth Microbiome Project Consortium.

Thursday, 26 October 2017 10:16
https://www.asm.org/index.php/podcasts/meet-the-microbiologist/item/6882-mtm-68
Microbiomes everywhere with Jack Gilbert - MTM 68

----------
## unclassified

https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/publicaton_metagenome_for-human-health-j.pdf
ヒトの健康における細菌およびメタゲノムイルミナテクノロジーを使用した最新論文の概要
https://jp.illumina.com/science/publication-reviews.html
論文集 | 最近発表された研究の要点

https://www.jstage.jst.go.jp/article/johokanri/55/3/55_167/_html/-char/ja
メタゲノム解析の現状と将来　知識データベースの開発
森 宙史, 山田 拓司, 黒川 顕
55 巻 (2012) 3 号 p. 167-175
さらに大規模なプロジェクトとして進行中なのが，2010年に日本を除く米欧中を中心とした12か国70人の研究者が立ち上げたEMP（Earth Microbiome Project）10)である。

山田拓司 著

https://www.jstage.jst.go.jp/article/jim/29/1/29_19/_article/-char/ja/
https://www.jstage.jst.go.jp/article/jim/29/1/29_19/_pdf
腸内細菌群集構造のメタゲノム解析 - J-Stage
腸内細菌学雑誌 29 : 19-22,2015

https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/2014_webinar_mktg30_microbiology.pdf
[PDF]ヒト腸内細菌叢メタゲノム解析

https://www.jstage.jst.go.jp/article/kagakutoseibutsu/51/12/51_802/_pdf
ヒト腸内メタゲノム解析が 広げる医療展開 - J-Stage
化学と生物 Vol. 51, No. 12, 2013

----------


2010年12月23日
http://www.nikkei-science.com/?p=14373
Science誌の選ぶこの10年の10大成果：マイクロバイオーム | 日経サイエンス







----------
### GWAS
- [Genome-wide association study (GWAS)](https://en.wikipedia.org/wiki/Genome-wide_association_study)
- [ゲノムワイド関連解析](https://bsd.neuroinf.jp/wiki/ゲノムワイド関連解析)
- [[PDF]次世代GWAS研究 - イルミナ株式会社](https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/primer_next_generation_gwas_research-j.pdf)



https://twitter.com/johnlees6/status/1013791082872745985
John Lees on Twitter: "Have a look at our (now published) short note about our microbial GWAS software pyseer: https://t.co/7u8ko59mb1 More info here: https://t.co/xOsJbzci33 We've also got some new features coming soon, how exciting"
10:26 AM - 2 Jul 2018

https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty539/5047751
pyseer: a comprehensive tool for microbial pangenome-wide association studies | Bioinformatics | Oxford Academic

http://pyseer.readthedocs.io/en/master/
pyseer documentation — pyseer 1.1.1 documentation


https://www.biorxiv.org/content/early/2018/02/15/266312
pyseer: a comprehensive tool for microbial pangenome-wide association studies | bioRxiv
- https://github.com/mgalardini/pyseer
- http://pyseer.readthedocs.io

6 Feb 2018
https://twitter.com/strnr/status/960873677532385280
Stephen Turner on Twitter: "treeWAS: A phylogenetic tree-based approach to genome-wide association studies in microbes code https://t.co/JprApEzBbj paper https://t.co/DZN9Jod0tQ docs https://t.co/e5aExAXb1L… https://t.co/ghmiArcTyI"

https://www.ncbi.nlm.nih.gov/pubmed/27840430
Nat Rev Genet. 2017 Jan;18(1):41-50. doi: 10.1038/nrg.2016.132. Epub 2016 Nov 14.
Microbial genome-wide association studies: lessons from human GWAS.
Power RA1, Parkhill J2, de Oliveira T1,2,3.

https://www.ncbi.nlm.nih.gov/pubmed/27887642
Genome Biol. 2016 Nov 25;17(1):238.
Rapid scoring of genes in microbial pan-genome-wide association studies with Scoary.
Brynildsrud O1, Bohlin J2, Scheffer L2,3, Eldholm V2.

https://www.ncbi.nlm.nih.gov/pubmed/27572652
Nat Microbiol. 2016 Apr 26;1:16059. doi: 10.1038/nmicrobiol.2016.59.
Bacterial genomics: Microbial GWAS coming of age.
Falush D1.

https://www.biostars.org/p/197270/
Is there any good GWAS software for the whole bacterial genomes?

https://www.ncbi.nlm.nih.gov/pubmed/27892531
Sci Rep. 2016 Nov 28;6:37811. doi: 10.1038/srep37811.
A genome-wide association study identifies a horizontally transferred bacterial surface adhesin gene associated with antimicrobial resistant strains.
Suzuki M1, Shibayama K1, Yahara K1.
http://www.aeplan.co.jp/sgmj2017/pdf/program_neminar.pdf
3O1-09
ゲノムワイド関連解析による水平伝達に由来しカルバペネム耐性株に広く保存された 細胞表面接着因子の発見

https://twitter.com/search?q=GWAS%20microbiome

https://www.ncbi.nlm.nih.gov/pubmed/28824167
Nat Rev Genet. 2017 Nov;18(11):690-699. doi: 10.1038/nrg.2017.63. Epub 2017 Aug 21.
Human genetic variation and the gut microbiome in disease.
Hall AB1,2, Tolonen AC1, Xavier RJ1,2,3,4.

https://www.ncbi.nlm.nih.gov/pubmed/27694959
Nat Genet. 2016 Nov;48(11):1407-1412. doi: 10.1038/ng.3663. Epub 2016 Oct 3.
The effect of host genetics on the gut microbiome.

https://www.ncbi.nlm.nih.gov/pubmed/26528553
PLoS One. 2015 Nov 3;10(11):e0140301. doi: 10.1371/journal.pone.0140301. eCollection 2015.
Genome-Wide Association Studies of the Human Gut Microbiota.
Davenport ER1, Cusanovich DA1, Michelini K1, Barreiro LB2, Ober C1, Gilad Y1.

2016年7月11日
http://first.lifesciencedb.jp/archives/12735
ゲノムワイド関連解析を用いたイネにおける農業形質にかかわる新規の遺伝子の同定

http://plaza.umin.ac.jp/~yokada/yokada.files/JSHG2012_YukinoriOkada_Award.pdf
2012/10/26 日本人類遺伝学会第57回大会 奨励賞受賞講演
ゲノムワイド関連解析を通じた疾患感受性遺伝子の同定

2009年3月26日
https://www.natureasia.com/ja-jp/jobs/tokushu/detail/154
ゲノムを巨視的にとらえる…GWASとエピジェネティクスの試み

2009年3月
https://ci.nii.ac.jp/els/contentscinii_20180212133359.pdf?id=ART0009161346
全ゲノム関連解析 (GWAS)の統計的手法
----------
## meta-analysis
https://ja.wikipedia.org/wiki/メタアナリシス

https://github.com/haruosuz/statistics#meta-analysis

https://twitter.com/search?f=tweets&q=microbiome%20meta-analysis

### almlab
http://almlab.mit.edu/

https://twitter.com/gibbological/status/988476547614105601
Sean Gibbons on Twitter: "Befuddled by batch effects? Check out our simple, non-parametric solution for 16S case-control studies in @PLOSCompBiol https://t.co/grMDm7SIQl -- method available as a @qiime2 plugin thanks to @cduvallet"

https://www.ncbi.nlm.nih.gov/pubmed/29684016
PLoS Comput Biol. 2018 Apr 23;14(4):e1006102. doi: 10.1371/journal.pcbi.1006102. eCollection 2018 Apr.
Correcting for batch effects in case-control microbiome studies.
Gibbons SM1,2,3, Duvallet C1,2, Alm EJ1,2,3.
微生物群集の症例対照研究におけるバッチ効果の補正

April 17, 2018
https://cduvallet.github.io/posts/2018/04/microbiome-data
An empirical analysis of microbiome data availability - Claire Duvallet

https://www.ncbi.nlm.nih.gov/pubmed/29209090
Nat Commun. 2017 Dec 5;8(1):1784. doi: 10.1038/s41467-017-01973-8.
Meta-analysis of gut microbiome studies identifies disease-specific and shared responses.
Duvallet C1,2, Gibbons SM1,2,3, Gurry T1,2,3, Irizarry RA4,5, Alm EJ6,7,8.
We perform a cross-disease meta-analysis of these studies using standardized methods. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5716994/
Code availability
The code to reproduce all of the analyses in this paper is available at https://github.com/cduvallet/microbiomeHD. 

https://twitter.com/Kazu_Kasahara/status/938469390739234816
Kazu_Kasahara on Twitter: "流行しているマイクロバイオーム研究に警鐘を鳴らす論文。28報の研究、10疾患のメタ解析から、病気で認める"dysbiosis"の多くは疾患特異的ではなく、バイオマーカーになり難い。 https://t.co/AVJHhcCCcI"
1:05 PM - 6 Dec 2017

### 2018-06
https://twitter.com/ayaKT/status/1005378142146457600
ayaKT on Twitter: "~19000人規模のmicrobiome-GWAS に着手しました論文。どんな解析結果が出てくるのか楽しみ。 いろんな16Sシーケンス手法を比較可能にする考えかたも、勉強になる。fmfm https://t.co/hNMXyZHzWj"
5:16 AM - 9 Jun 2018

https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0479-3
Meta-analysis of human genome-microbiome association studies: the MiBioGen consortium initiative | Microbiome
We are now in the process of benchmarking the association tests and performing meta-analyses of genome-wide associations. All pipeline and summary statistics results will be shared using public data repositories.

### 2017-12-22
https://www.ncbi.nlm.nih.gov/pubmed/29273717
Nat Commun. 2017 Dec 22;8(1):2260. doi: 10.1038/s41467-017-02209-5.
Strain profiling and epidemiology of bacterial species from metagenomic sequencing.
Albanese D1, Donati C2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5741664/
- Here we present StrainEst, a novel, reference-based method that uses the single-nucleotide variants (SNV) profiles of the available genomes of selected species to determine the number and identity of coexisting strains and their relative abundances in mixed metagenomic samples.
- meta-analyses

### 2017-11-20
https://twitter.com/NatureMicrobiol/status/932704694912389120
Nature Microbiology on Twitter: ".@drkellysierra & co: Integration and meta-analysis of global #soil datasets uncover patterns of #microbial structure,… https://t.co/QB51kFBtK5"
3:18 PM - 20 Nov 2017

https://www.nature.com/articles/s41564-017-0062-x
Detecting macroecological patterns in bacterial communities across independent studies of global soils | Nature Microbiology
- Whereas previous meta-analysis efforts have focused on bacterial diversity measures or abundances of major taxa, we show that disparate amplicon sequence data can be combined at the taxonomy-based level to assess bacterial community structure.

### 2016
https://www.ncbi.nlm.nih.gov/pubmed/27400279
PLoS Comput Biol. 2016 Jul 11;12(7):e1004977. doi: 10.1371/journal.pcbi.1004977. eCollection 2016 Jul.
Machine Learning Meta-analysis of Large Metagenomic Datasets: Tools and Biological Insights.
Pasolli E1, Truong DT1, Malik F2, Waldron L2, Segata N1.
http://segatalab.cibio.unitn.it/tools/metaml
MetAML - Metagenomic prediction Analysis based on Machine Learning
機械学習

### 2015
https://www.ncbi.nlm.nih.gov/pubmed/26459172
Microbiome. 2015 Oct 13;3:49. doi: 10.1186/s40168-015-0108-3.
Microbiota of the indoor environment: a meta-analysis.
Adams RI1, Bateman AC2, Bik HM3,4, Meadow JF5.
室内環境の微生物群集：メタアナリシス

----------

